.storybook/docs-root.css

```diff
@@ -1,24 +1,26 @@
-#docs-root .sbdocs-content {
+#storybook-docs .sbdocs-content {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32018&commitId=569874be161aa314bac16c9ce333dc3985534abf&env=prod&clientType=FLUENTUI)


  <details><summary><b>(Dev-Only) Suggestions</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.chromium.png|0| Added|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.Hovering over a wide suggestion element.chromium.png|0| Removed|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.Hovering over the X button on a wide suggestion element.chromium.png|0| Removed|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.default.chromium.png|0| Removed|

</details>


  <details><summary><b>ActivityItem</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Compact.chromium.png|0| Added|
|ActivityItem.CompactPersonas.chromium.png|0| Added|
|ActivityItem.Personas RTL.chromium.png|0| Added|
|ActivityItem.Personas.chromium.png|0| Added|
|ActivityItem.Root RTL.chromium.png|0| Added|
|ActivityItem.Root.chromium.png|0| Added|
|ActivityItem.Compact.default.chromium.png|0| Removed|
|ActivityItem.CompactPersonas.default.chromium.png|0| Removed|
|ActivityItem.Personas - RTL.default.chromium.png|0| Removed|
|ActivityItem.Personas.default.chromium.png|0| Removed|
|ActivityItem.Root - RTL.default.chromium.png|0| Removed|
|ActivityItem.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Breadcrumb</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb.Button RTL.chromium.png|0| Added|
|Breadcrumb.Button.chromium.png|0| Added|
|Breadcrumb.Root RTL.chromium.png|0| Added|
|Breadcrumb.Root.chromium.png|0| Added|
|Breadcrumb.Button - RTL.click.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.default.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.hover.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.itemLinkFocus.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.overflowButtonFocus.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.shortTitleHover.chromium.png|0| Removed|
|Breadcrumb.Button.click.chromium.png|0| Removed|
|Breadcrumb.Button.default.chromium.png|0| Removed|
|Breadcrumb.Button.hover.chromium.png|0| Removed|
|Breadcrumb.Button.itemLinkFocus.chromium.png|0| Removed|
|Breadcrumb.Button.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Button.overflowButtonFocus.chromium.png|0| Removed|
|Breadcrumb.Button.shortTitleHover.chromium.png|0| Removed|
|Breadcrumb.Hovering items.actionable hover.chromium.png|0| Removed|
|Breadcrumb.Hovering items.actionable overflow hover.chromium.png|0| Removed|
|Breadcrumb.Hovering items.non-actionable hover.chromium.png|0| Removed|
|Breadcrumb.Hovering items.non-actionable overflow hover.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.click.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.default.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.hover.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.itemLinkFocus.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.overflowButtonFocus.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.shortTitleHover.chromium.png|0| Removed|
|Breadcrumb.Root.click.chromium.png|0| Removed|
|Breadcrumb.Root.default.chromium.png|0| Removed|
|Breadcrumb.Root.hover.chromium.png|0| Removed|
|Breadcrumb.Root.itemLinkFocus.chromium.png|0| Removed|
|Breadcrumb.Root.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Root.overflowButtonFocus.chromium.png|0| Removed|
|Breadcrumb.Root.shortTitleHover.chromium.png|0| Removed|

</details>


  <details><summary><b>Button (compat)</b> 40 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button (compat).Checked.chromium.png|0| Added|
|Button (compat).Disabled.chromium.png|0| Added|
|Button (compat).Icon Only.chromium.png|0| Added|
|Button (compat).No Icon.chromium.png|0| Added|
|Button (compat).Primary Anchor.chromium.png|0| Added|
|Button (compat).Primary Checked.chromium.png|0| Added|
|Button (compat).Primary Disabled.chromium.png|0| Added|
|Button (compat).Primary.chromium.png|0| Added|
|Button (compat).Root RTL.chromium.png|0| Added|
|Button (compat).Root.chromium.png|0| Added|
|Button (compat).Checked.default.chromium.png|0| Removed|
|Button (compat).Checked.hover.chromium.png|0| Removed|
|Button (compat).Checked.pressed.chromium.png|0| Removed|
|Button (compat).Disabled.default.chromium.png|0| Removed|
|Button (compat).Disabled.hover.chromium.png|0| Removed|
|Button (compat).Disabled.pressed.chromium.png|0| Removed|
|Button (compat).Icon Only.default.chromium.png|0| Removed|
|Button (compat).Icon Only.hover.chromium.png|0| Removed|
|Button (compat).Icon Only.pressed.chromium.png|0| Removed|
|Button (compat).No Icon.default.chromium.png|0| Removed|
|Button (compat).No Icon.hover.chromium.png|0| Removed|
|Button (compat).No Icon.pressed.chromium.png|0| Removed|
|Button (compat).Primary Anchor.default.chromium.png|0| Removed|
|Button (compat).Primary Anchor.hover.chromium.png|0| Removed|
|Button (compat).Primary Anchor.pressed.chromium.png|0| Removed|
|Button (compat).Primary Checked.default.chromium.png|0| Removed|
|Button (compat).Primary Checked.hover.chromium.png|0| Removed|
|Button (compat).Primary Checked.pressed.chromium.png|0| Removed|
|Button (compat).Primary Disabled.default.chromium.png|0| Removed|
|Button (compat).Primary Disabled.hover.chromium.png|0| Removed|
|Button (compat).Primary Disabled.pressed.chromium.png|0| Removed|
|Button (compat).Primary.default.chromium.png|0| Removed|
|Button (compat).Primary.hover.chromium.png|0| Removed|
|Button (compat).Primary.pressed.chromium.png|0| Removed|
|Button (compat).Root - RTL.default.chromium.png|0| Removed|
|Button (compat).Root - RTL.hover.chromium.png|0| Removed|
|Button (compat).Root - RTL.pressed.chromium.png|0| Removed|
|Button (compat).Root.default.chromium.png|0| Removed|
|Button (compat).Root.hover.chromium.png|0| Removed|
|Button (compat).Root.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Button Action (compat)</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Action (compat).Checked.chromium.png|0| Added|
|Button Action (compat).Disabled.chromium.png|0| Added|
|Button Action (compat).Icon Only.chromium.png|0| Added|
|Button Action (compat).No Icon.chromium.png|0| Added|
|Button Action (compat).Root RTL.chromium.png|0| Added|
|Button Action (compat).Root.chromium.png|0| Added|
|Button Action (compat).Checked.default.chromium.png|0| Removed|
|Button Action (compat).Checked.hover.chromium.png|0| Removed|
|Button Action (compat).Checked.pressed.chromium.png|0| Removed|
|Button Action (compat).Disabled.default.chromium.png|0| Removed|
|Button Action (compat).Disabled.hover.chromium.png|0| Removed|
|Button Action (compat).Disabled.pressed.chromium.png|0| Removed|
|Button Action (compat).Icon Only.default.chromium.png|0| Removed|
|Button Action (compat).Icon Only.hover.chromium.png|0| Removed|
|Button Action (compat).Icon Only.pressed.chromium.png|0| Removed|
|Button Action (compat).No Icon.default.chromium.png|0| Removed|
|Button Action (compat).No Icon.hover.chromium.png|0| Removed|
|Button Action (compat).No Icon.pressed.chromium.png|0| Removed|
|Button Action (compat).Root - RTL.default.chromium.png|0| Removed|
|Button Action (compat).Root - RTL.hover.chromium.png|0| Removed|
|Button Action (compat).Root - RTL.pressed.chromium.png|0| Removed|
|Button Action (compat).Root.default.chromium.png|0| Removed|
|Button Action (compat).Root.hover.chromium.png|0| Removed|
|Button Action (compat).Root.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Button Command (compat)</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Command (compat).Checked.chromium.png|0| Added|
|Button Command (compat).Disabled.chromium.png|0| Added|
|Button Command (compat).Root RTL.chromium.png|0| Added|
|Button Command (compat).Root.chromium.png|0| Added|
|Button Command (compat).Checked.default.chromium.png|0| Removed|
|Button Command (compat).Checked.hover.chromium.png|0| Removed|
|Button Command (compat).Checked.open.chromium.png|0| Removed|
|Button Command (compat).Checked.pressed.chromium.png|0| Removed|
|Button Command (compat).Disabled.default.chromium.png|0| Removed|
|Button Command (compat).Disabled.hover.chromium.png|0| Removed|
|Button Command (compat).Disabled.open.chromium.png|0| Removed|
|Button Command (compat).Disabled.pressed.chromium.png|0| Removed|
|Button Command (compat).Root - RTL.default.chromium.png|0| Removed|
|Button Command (compat).Root - RTL.hover.chromium.png|0| Removed|
|Button Command (compat).Root - RTL.open.chromium.png|0| Removed|
|Button Command (compat).Root - RTL.pressed.chromium.png|0| Removed|
|Button Command (compat).Root.default.chromium.png|0| Removed|
|Button Command (compat).Root.hover.chromium.png|0| Removed|
|Button Command (compat).Root.open.chromium.png|0| Removed|
|Button Command (compat).Root.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Button Compound (compat)</b> 32 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Compound (compat).Checked.chromium.png|0| Added|
|Button Compound (compat).Disabled.chromium.png|0| Added|
|Button Compound (compat).No Icon.chromium.png|0| Added|
|Button Compound (compat).Primary Checked.chromium.png|0| Added|
|Button Compound (compat).Primary Disabled.chromium.png|0| Added|
|Button Compound (compat).Primary.chromium.png|0| Added|
|Button Compound (compat).Root RTL.chromium.png|0| Added|
|Button Compound (compat).Root.chromium.png|0| Added|
|Button Compound (compat).Checked.default.chromium.png|0| Removed|
|Button Compound (compat).Checked.hover.chromium.png|0| Removed|
|Button Compound (compat).Checked.pressed.chromium.png|0| Removed|
|Button Compound (compat).Disabled.default.chromium.png|0| Removed|
|Button Compound (compat).Disabled.hover.chromium.png|0| Removed|
|Button Compound (compat).Disabled.pressed.chromium.png|0| Removed|
|Button Compound (compat).No Icon.default.chromium.png|0| Removed|
|Button Compound (compat).No Icon.hover.chromium.png|0| Removed|
|Button Compound (compat).No Icon.pressed.chromium.png|0| Removed|
|Button Compound (compat).Primary Checked.default.chromium.png|0| Removed|
|Button Compound (compat).Primary Checked.hover.chromium.png|0| Removed|
|Button Compound (compat).Primary Checked.pressed.chromium.png|0| Removed|
|Button Compound (compat).Primary Disabled.default.chromium.png|0| Removed|
|Button Compound (compat).Primary Disabled.hover.chromium.png|0| Removed|
|Button Compound (compat).Primary Disabled.pressed.chromium.png|0| Removed|
|Button Compound (compat).Primary.default.chromium.png|0| Removed|
|Button Compound (compat).Primary.hover.chromium.png|0| Removed|
|Button Compound (compat).Primary.pressed.chromium.png|0| Removed|
|Button Compound (compat).Root - RTL.default.chromium.png|0| Removed|
|Button Compound (compat).Root - RTL.hover.chromium.png|0| Removed|
|Button Compound (compat).Root - RTL.pressed.chromium.png|0| Removed|
|Button Compound (compat).Root.default.chromium.png|0| Removed|
|Button Compound (compat).Root.hover.chromium.png|0| Removed|
|Button Compound (compat).Root.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Button Special Scenarios (compat)</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Special Scenarios (compat).no flex shrink.chromium.png|0| Added|
|Button Special Scenarios (compat).primary with placeholder.chromium.png|0| Added|
|Button Special Scenarios (compat).no flex shrink.default.chromium.png|0| Removed|
|Button Special Scenarios (compat).primary with placeholder.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Button Split (compat)</b> 40 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Split (compat).Checked.chromium.png|0| Added|
|Button Split (compat).Command Split.chromium.png|0| Added|
|Button Split (compat).Default with Primary Action Disabled.chromium.png|0| Added|
|Button Split (compat).Disabled.chromium.png|0| Added|
|Button Split (compat).Primary Checked.chromium.png|0| Added|
|Button Split (compat).Primary Disabled.chromium.png|0| Added|
|Button Split (compat).Primary with Primary Action Disabled.chromium.png|0| Added|
|Button Split (compat).Primary.chromium.png|0| Added|
|Button Split (compat).Root RTL.chromium.png|0| Added|
|Button Split (compat).Root.chromium.png|0| Added|
|Button Split (compat).Checked.default.chromium.png|0| Removed|
|Button Split (compat).Checked.hover main.chromium.png|0| Removed|
|Button Split (compat).Checked.hover split.chromium.png|0| Removed|
|Button Split (compat).Command Split.default.chromium.png|0| Removed|
|Button Split (compat).Command Split.hover main.chromium.png|0| Removed|
|Button Split (compat).Command Split.hover split.chromium.png|0| Removed|
|Button Split (compat).Default with Primary Action Disabled.default.chromium.png|0| Removed|
|Button Split (compat).Default with Primary Action Disabled.hover main.chromium.png|0| Removed|
|Button Split (compat).Default with Primary Action Disabled.hover split.chromium.png|0| Removed|
|Button Split (compat).Disabled.default.chromium.png|0| Removed|
|Button Split (compat).Disabled.hover main.chromium.png|0| Removed|
|Button Split (compat).Disabled.hover split.chromium.png|0| Removed|
|Button Split (compat).Primary Checked.default.chromium.png|0| Removed|
|Button Split (compat).Primary Checked.hover main.chromium.png|0| Removed|
|Button Split (compat).Primary Checked.hover split.chromium.png|0| Removed|
|Button Split (compat).Primary Disabled.default.chromium.png|0| Removed|
|Button Split (compat).Primary Disabled.hover main.chromium.png|0| Removed|
|Button Split (compat).Primary Disabled.hover split.chromium.png|0| Removed|
|Button Split (compat).Primary with Primary Action Disabled.default.chromium.png|0| Removed|
|Button Split (compat).Primary with Primary Action Disabled.hover main.chromium.png|0| Removed|
|Button Split (compat).Primary with Primary Action Disabled.hover split.chromium.png|0| Removed|
|Button Split (compat).Primary.default.chromium.png|0| Removed|
|Button Split (compat).Primary.hover main.chromium.png|0| Removed|
|Button Split (compat).Primary.hover split.chromium.png|0| Removed|
|Button Split (compat).Root - RTL.default.chromium.png|0| Removed|
|Button Split (compat).Root - RTL.hover main.chromium.png|0| Removed|
|Button Split (compat).Root - RTL.hover split.chromium.png|0| Removed|
|Button Split (compat).Root.default.chromium.png|0| Removed|
|Button Split (compat).Root.hover main.chromium.png|0| Removed|
|Button Split (compat).Root.hover split.chromium.png|0| Removed|

</details>


  <details><summary><b>Calendar</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar.Root RTL.chromium.png|0| Added|
|Calendar.Root.chromium.png|0| Added|
|Calendar.Root - RTL.default.chromium.png|0| Removed|
|Calendar.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Calendar - No Month Option</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar - No Month Option.Show Month as Overlay and no Go To Today.chromium.png|0| Added|
|Calendar - No Month Option.Show Month selection type.chromium.png|0| Added|
|Calendar - No Month Option.Show Week selection type.chromium.png|0| Added|
|Calendar - No Month Option.Show Work Week selection type.chromium.png|0| Added|
|Calendar - No Month Option.Show Month as Overlay and no Go To Today.default.chromium.png|0| Removed|
|Calendar - No Month Option.Show Month selection type.default.chromium.png|0| Removed|
|Calendar - No Month Option.Show Week selection type.default.chromium.png|0| Removed|
|Calendar - No Month Option.Show Work Week selection type.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Callout</b> 44 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Beak 25.chromium.png|0| Added|
|Callout.Bottom Left Edge RTL.chromium.png|0| Added|
|Callout.Bottom Right Edge RTL.chromium.png|0| Added|
|Callout.Bottom auto edge.chromium.png|0| Added|
|Callout.Bottom center.chromium.png|0| Added|
|Callout.Bottom left edge.chromium.png|0| Added|
|Callout.Bottom right edge.chromium.png|0| Added|
|Callout.Gap space 25.chromium.png|0| Added|
|Callout.Left bottom edge.chromium.png|0| Added|
|Callout.Left center.chromium.png|0| Added|
|Callout.Left top edge.chromium.png|0| Added|
|Callout.No beak.chromium.png|0| Added|
|Callout.No callout width specified.chromium.png|0| Added|
|Callout.Rendering callout attached to a rectangle.chromium.png|0| Added|
|Callout.Right bottom edge.chromium.png|0| Added|
|Callout.Right center.chromium.png|0| Added|
|Callout.Right top edge.chromium.png|0| Added|
|Callout.Root.chromium.png|0| Added|
|Callout.Top auto edge.chromium.png|0| Added|
|Callout.Top center.chromium.png|0| Added|
|Callout.Top left edge.chromium.png|0| Added|
|Callout.Top right edge.chromium.png|0| Added|
|Callout.Beak 25.default.chromium.png|0| Removed|
|Callout.Bottom auto edge.default.chromium.png|0| Removed|
|Callout.Bottom center.default.chromium.png|0| Removed|
|Callout.Bottom left edge - RTL.default.chromium.png|0| Removed|
|Callout.Bottom left edge.default.chromium.png|0| Removed|
|Callout.Bottom right edge - RTL.default.chromium.png|0| Removed|
|Callout.Bottom right edge.default.chromium.png|0| Removed|
|Callout.Gap space 25.default.chromium.png|0| Removed|
|Callout.Left bottom edge.default.chromium.png|0| Removed|
|Callout.Left center.default.chromium.png|0| Removed|
|Callout.Left top edge.default.chromium.png|0| Removed|
|Callout.No beak.default.chromium.png|0| Removed|
|Callout.No callout width specified.default.chromium.png|0| Removed|
|Callout.Rendering callout attached to a rectangle.default.chromium.png|0| Removed|
|Callout.Right bottom edge.default.chromium.png|0| Removed|
|Callout.Right center.default.chromium.png|0| Removed|
|Callout.Right top edge.default.chromium.png|0| Removed|
|Callout.Root.default.chromium.png|0| Removed|
|Callout.Top auto edge.default.chromium.png|0| Removed|
|Callout.Top center.default.chromium.png|0| Removed|
|Callout.Top left edge.default.chromium.png|0| Removed|
|Callout.Top right edge.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Checkbox</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checked disabled.chromium.png|0| Added|
|Checkbox.Checked.chromium.png|0| Added|
|Checkbox.Controlled Indeterminate disabled.chromium.png|0| Added|
|Checkbox.Controlled Indeterminate.chromium.png|0| Added|
|Checkbox.Custom render Checkbox.chromium.png|0| Added|
|Checkbox.End RTL.chromium.png|0| Added|
|Checkbox.End.chromium.png|0| Added|
|Checkbox.Multi-line Checkbox.chromium.png|0| Added|
|Checkbox.Unchecked RTL.chromium.png|0| Added|
|Checkbox.Unchecked disabled.chromium.png|0| Added|
|Checkbox.Unchecked.chromium.png|0| Added|
|Checkbox.Uncontrolled Indeterminate disabled.chromium.png|0| Added|
|Checkbox.Uncontrolled Indeterminate.chromium.png|0| Added|
|Checkbox.Checked disabled.default.chromium.png|0| Removed|
|Checkbox.Checked.default.chromium.png|0| Removed|
|Checkbox.Controlled Indeterminate disabled.default.chromium.png|0| Removed|
|Checkbox.Controlled Indeterminate.default.chromium.png|0| Removed|
|Checkbox.Custom render Checkbox.default.chromium.png|0| Removed|
|Checkbox.End - RTL.default.chromium.png|0| Removed|
|Checkbox.End.default.chromium.png|0| Removed|
|Checkbox.Multi-line Checkbox.default.chromium.png|0| Removed|
|Checkbox.Unchecked - RTL.default.chromium.png|0| Removed|
|Checkbox.Unchecked disabled.default.chromium.png|0| Removed|
|Checkbox.Unchecked.default.chromium.png|0| Removed|
|Checkbox.Uncontrolled Indeterminate disabled.default.chromium.png|0| Removed|
|Checkbox.Uncontrolled Indeterminate.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Checkbox Indeterminate</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.chromium.png|0| Added|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.chromium.png|0| Added|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.clicked.chromium.png|0| Removed|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.default.chromium.png|0| Removed|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.hover.chromium.png|0| Removed|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.clicked.chromium.png|0| Removed|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.default.chromium.png|0| Removed|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.hover.chromium.png|0| Removed|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.clicked.chromium.png|0| Removed|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.default.chromium.png|0| Removed|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.hover.chromium.png|0| Removed|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.clicked.chromium.png|0| Removed|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.default.chromium.png|0| Removed|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>ChoiceGroup</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ChoiceGroup.Required.chromium.png|0| Added|
|ChoiceGroup.Root.chromium.png|0| Added|
|ChoiceGroup.With Icons RTL.chromium.png|0| Added|
|ChoiceGroup.With default size images.chromium.png|0| Added|
|ChoiceGroup.With icons.chromium.png|0| Added|
|ChoiceGroup.With large size images.chromium.png|0| Added|
|ChoiceGroup.Required.default.chromium.png|0| Removed|
|ChoiceGroup.Required.hover selected.chromium.png|0| Removed|
|ChoiceGroup.Required.hover unselected.chromium.png|0| Removed|
|ChoiceGroup.Required.selected.chromium.png|0| Removed|
|ChoiceGroup.Root.default.chromium.png|0| Removed|
|ChoiceGroup.Root.hover selected.chromium.png|0| Removed|
|ChoiceGroup.Root.hover unselected.chromium.png|0| Removed|
|ChoiceGroup.Root.selected.chromium.png|0| Removed|
|ChoiceGroup.With default size images.default.chromium.png|0| Removed|
|ChoiceGroup.With default size images.hover selected.chromium.png|0| Removed|
|ChoiceGroup.With default size images.hover unselected.chromium.png|0| Removed|
|ChoiceGroup.With default size images.selected.chromium.png|0| Removed|
|ChoiceGroup.With icons - RTL.default.chromium.png|0| Removed|
|ChoiceGroup.With icons - RTL.hover selected.chromium.png|0| Removed|
|ChoiceGroup.With icons - RTL.hover unselected.chromium.png|0| Removed|
|ChoiceGroup.With icons - RTL.selected.chromium.png|0| Removed|
|ChoiceGroup.With icons.default.chromium.png|0| Removed|
|ChoiceGroup.With icons.hover selected.chromium.png|0| Removed|
|ChoiceGroup.With icons.hover unselected.chromium.png|0| Removed|
|ChoiceGroup.With icons.selected.chromium.png|0| Removed|
|ChoiceGroup.With large size images.default.chromium.png|0| Removed|
|ChoiceGroup.With large size images.hover selected.chromium.png|0| Removed|
|ChoiceGroup.With large size images.hover unselected.chromium.png|0| Removed|
|ChoiceGroup.With large size images.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>Coachmark</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.chromium.png|0| Added|
|Coachmark.Rendering Coachmark attached to a rectangle.chromium.png|0| Added|
|Coachmark.Collapsed.default.chromium.png|0| Removed|
|Coachmark.Rendering Coachmark attached to a rectangle.default.chromium.png|0| Removed|

</details>


  <details><summary><b>ColorPicker</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ColorPicker.Blue.chromium.png|0| Added|
|ColorPicker.Root RTL.chromium.png|0| Added|
|ColorPicker.Root.chromium.png|0| Added|
|ColorPicker.Blue.default.chromium.png|0| Removed|
|ColorPicker.Root - RTL.default.chromium.png|0| Removed|
|ColorPicker.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>ComboBox</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ComboBox.Disabled.chromium.png|0| Added|
|ComboBox.Error.chromium.png|0| Added|
|ComboBox.Placeholder.chromium.png|0| Added|
|ComboBox.Root RTL.chromium.png|0| Added|
|ComboBox.Root.chromium.png|0| Added|
|ComboBox.Styled.chromium.png|0| Added|
|ComboBox.Disabled.click.chromium.png|0| Removed|
|ComboBox.Disabled.default.chromium.png|0| Removed|
|ComboBox.Disabled.hover.chromium.png|0| Removed|
|ComboBox.Error.click.chromium.png|0| Removed|
|ComboBox.Error.default.chromium.png|0| Removed|
|ComboBox.Error.hover.chromium.png|0| Removed|
|ComboBox.Placeholder.click.chromium.png|0| Removed|
|ComboBox.Placeholder.default.chromium.png|0| Removed|
|ComboBox.Placeholder.hover.chromium.png|0| Removed|
|ComboBox.Root - RTL.click.chromium.png|0| Removed|
|ComboBox.Root - RTL.default.chromium.png|0| Removed|
|ComboBox.Root - RTL.hover.chromium.png|0| Removed|
|ComboBox.Root.click.chromium.png|0| Removed|
|ComboBox.Root.default.chromium.png|0| Removed|
|ComboBox.Root.hover.chromium.png|0| Removed|
|ComboBox.Styled.click.chromium.png|0| Removed|
|ComboBox.Styled.default.chromium.png|0| Removed|
|ComboBox.Styled.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>CommandBar</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CommandBar.Icons only.chromium.png|0| Added|
|CommandBar.Root RTL.chromium.png|0| Added|
|CommandBar.Root.chromium.png|0| Added|
|CommandBar.Text only.chromium.png|0| Added|
|CommandBar.Icons only.click.chromium.png|0| Removed|
|CommandBar.Icons only.default.chromium.png|0| Removed|
|CommandBar.Icons only.hover.chromium.png|0| Removed|
|CommandBar.Root - RTL.click.chromium.png|0| Removed|
|CommandBar.Root - RTL.default.chromium.png|0| Removed|
|CommandBar.Root - RTL.hover.chromium.png|0| Removed|
|CommandBar.Root.click.chromium.png|0| Removed|
|CommandBar.Root.default.chromium.png|0| Removed|
|CommandBar.Root.hover.chromium.png|0| Removed|
|CommandBar.Text only.click.chromium.png|0| Removed|
|CommandBar.Text only.default.chromium.png|0| Removed|
|CommandBar.Text only.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>ContextualMenu</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ContextualMenu.Root.chromium.png|0| Added|
|ContextualMenu.With Secondary Text RTL.chromium.png|0| Added|
|ContextualMenu.With Submenu RTL.chromium.png|0| Added|
|ContextualMenu.With headers.chromium.png|0| Added|
|ContextualMenu.With icons.chromium.png|0| Added|
|ContextualMenu.With secondaryText.chromium.png|0| Added|
|ContextualMenu.With split button submenu.chromium.png|0| Added|
|ContextualMenu.With submenu.chromium.png|0| Added|
|ContextualMenu.Root.click.chromium.png|0| Removed|
|ContextualMenu.Root.default.chromium.png|0| Removed|
|ContextualMenu.Root.hover.chromium.png|0| Removed|
|ContextualMenu.With headers.click.chromium.png|0| Removed|
|ContextualMenu.With headers.default.chromium.png|0| Removed|
|ContextualMenu.With headers.hover.chromium.png|0| Removed|
|ContextualMenu.With icons.click.chromium.png|0| Removed|
|ContextualMenu.With icons.default.chromium.png|0| Removed|
|ContextualMenu.With icons.hover.chromium.png|0| Removed|
|ContextualMenu.With secondaryText - RTL.click.chromium.png|0| Removed|
|ContextualMenu.With secondaryText - RTL.default.chromium.png|0| Removed|
|ContextualMenu.With secondaryText - RTL.hover.chromium.png|0| Removed|
|ContextualMenu.With secondaryText.click.chromium.png|0| Removed|
|ContextualMenu.With secondaryText.default.chromium.png|0| Removed|
|ContextualMenu.With secondaryText.hover.chromium.png|0| Removed|
|ContextualMenu.With split button submenu.click.chromium.png|0| Removed|
|ContextualMenu.With split button submenu.default.chromium.png|0| Removed|
|ContextualMenu.With split button submenu.hover.chromium.png|0| Removed|
|ContextualMenu.With submenu - RTL.click.chromium.png|0| Removed|
|ContextualMenu.With submenu - RTL.default.chromium.png|0| Removed|
|ContextualMenu.With submenu - RTL.hover.chromium.png|0| Removed|
|ContextualMenu.With submenu.click.chromium.png|0| Removed|
|ContextualMenu.With submenu.default.chromium.png|0| Removed|
|ContextualMenu.With submenu.hover.chromium.png|0| Removed|
|ContextualMenu.With submenus with hrefs.item1 hovered.chromium.png|0| Removed|
|ContextualMenu.With submenus with hrefs.item2 hovered.chromium.png|0| Removed|
|ContextualMenu.With submenus with hrefs.menu opened.chromium.png|0| Removed|
|ContextualMenu.With submenus with hrefs.parent hovered.chromium.png|0| Removed|

</details>


  <details><summary><b>DatePicker</b> 42 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker.Allow text input.chromium.png|0| Added|
|DatePicker.Placeholder.chromium.png|0| Added|
|DatePicker.Required.chromium.png|0| Added|
|DatePicker.Root RTL.chromium.png|0| Added|
|DatePicker.Root.chromium.png|0| Added|
|DatePicker.Underlined and Required.chromium.png|0| Added|
|DatePicker.Underlined.chromium.png|0| Added|
|DatePicker.Allow text input.click.chromium.png|0| Removed|
|DatePicker.Allow text input.default.chromium.png|0| Removed|
|DatePicker.Allow text input.hover datepicker.chromium.png|0| Removed|
|DatePicker.Allow text input.hover day.chromium.png|0| Removed|
|DatePicker.Allow text input.hover month.chromium.png|0| Removed|
|DatePicker.Placeholder.click.chromium.png|0| Removed|
|DatePicker.Placeholder.default.chromium.png|0| Removed|
|DatePicker.Placeholder.hover datepicker.chromium.png|0| Removed|
|DatePicker.Placeholder.hover day.chromium.png|0| Removed|
|DatePicker.Placeholder.hover month.chromium.png|0| Removed|
|DatePicker.Required.click.chromium.png|0| Removed|
|DatePicker.Required.default.chromium.png|0| Removed|
|DatePicker.Required.hover datepicker.chromium.png|0| Removed|
|DatePicker.Required.hover day.chromium.png|0| Removed|
|DatePicker.Required.hover month.chromium.png|0| Removed|
|DatePicker.Root - RTL.click.chromium.png|0| Removed|
|DatePicker.Root - RTL.default.chromium.png|0| Removed|
|DatePicker.Root - RTL.hover datepicker.chromium.png|0| Removed|
|DatePicker.Root - RTL.hover day.chromium.png|0| Removed|
|DatePicker.Root - RTL.hover month.chromium.png|0| Removed|
|DatePicker.Root.click.chromium.png|0| Removed|
|DatePicker.Root.default.chromium.png|0| Removed|
|DatePicker.Root.hover datepicker.chromium.png|0| Removed|
|DatePicker.Root.hover day.chromium.png|0| Removed|
|DatePicker.Root.hover month.chromium.png|0| Removed|
|DatePicker.Underlined and Required.click.chromium.png|0| Removed|
|DatePicker.Underlined and Required.default.chromium.png|0| Removed|
|DatePicker.Underlined and Required.hover datepicker.chromium.png|0| Removed|
|DatePicker.Underlined and Required.hover day.chromium.png|0| Removed|
|DatePicker.Underlined and Required.hover month.chromium.png|0| Removed|
|DatePicker.Underlined.click.chromium.png|0| Removed|
|DatePicker.Underlined.default.chromium.png|0| Removed|
|DatePicker.Underlined.hover datepicker.chromium.png|0| Removed|
|DatePicker.Underlined.hover day.chromium.png|0| Removed|
|DatePicker.Underlined.hover month.chromium.png|0| Removed|

</details>


  <details><summary><b>DatePicker - Disabled</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker - Disabled.With Label.chromium.png|0| Added|
|DatePicker - Disabled.Without Label.chromium.png|0| Added|
|DatePicker - Disabled.Without Value.chromium.png|0| Added|
|DatePicker - Disabled.With Label.click.chromium.png|0| Removed|
|DatePicker - Disabled.With Label.default.chromium.png|0| Removed|
|DatePicker - Disabled.With Label.hover datepicker.chromium.png|0| Removed|
|DatePicker - Disabled.Without Label.click.chromium.png|0| Removed|
|DatePicker - Disabled.Without Label.default.chromium.png|0| Removed|
|DatePicker - Disabled.Without Label.hover datepicker.chromium.png|0| Removed|
|DatePicker - Disabled.Without Value.click.chromium.png|0| Removed|
|DatePicker - Disabled.Without Value.default.chromium.png|0| Removed|
|DatePicker - Disabled.Without Value.hover datepicker.chromium.png|0| Removed|

</details>


  <details><summary><b>DatePicker - No Month Option</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.chromium.png|0| Added|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.click.chromium.png|0| Removed|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.default.chromium.png|0| Removed|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.hover datepicker.chromium.png|0| Removed|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.hover day.chromium.png|0| Removed|

</details>


  <details><summary><b>DetailsHeader</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DetailsHeader.Root.chromium.png|0| Added|
|DetailsHeader.Root.borderWhileDragging.chromium.png|0| Removed|
|DetailsHeader.Root.default.chromium.png|0| Removed|
|DetailsHeader.Root.dropHint.chromium.png|0| Removed|
|DetailsHeader.Root.hoverDraggable.chromium.png|0| Removed|
|DetailsHeader.Root.hoverFrozenFirst.chromium.png|0| Removed|
|DetailsHeader.Root.hoverFrozenLast.chromium.png|0| Removed|

</details>


  <details><summary><b>DetailsList</b> 28 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DetailsList.Checkbox Visible Always RTL.chromium.png|0| Added|
|DetailsList.Checkbox Visible Always.chromium.png|0| Added|
|DetailsList.Compact.chromium.png|0| Added|
|DetailsList.Grouped with Checkbox Hidden.chromium.png|0| Added|
|DetailsList.Grouped.chromium.png|0| Added|
|DetailsList.Root.chromium.png|0| Added|
|DetailsList.Single Selection Mode.chromium.png|0| Added|
|DetailsList.Checkbox Visible Always - RTL.click.chromium.png|0| Removed|
|DetailsList.Checkbox Visible Always - RTL.default.chromium.png|0| Removed|
|DetailsList.Checkbox Visible Always - RTL.hover.chromium.png|0| Removed|
|DetailsList.Checkbox Visible Always.click.chromium.png|0| Removed|
|DetailsList.Checkbox Visible Always.default.chromium.png|0| Removed|
|DetailsList.Checkbox Visible Always.hover.chromium.png|0| Removed|
|DetailsList.Compact.click.chromium.png|0| Removed|
|DetailsList.Compact.default.chromium.png|0| Removed|
|DetailsList.Compact.hover.chromium.png|0| Removed|
|DetailsList.Grouped with Checkbox Hidden.click.chromium.png|0| Removed|
|DetailsList.Grouped with Checkbox Hidden.default.chromium.png|0| Removed|
|DetailsList.Grouped with Checkbox Hidden.hover.chromium.png|0| Removed|
|DetailsList.Grouped.click.chromium.png|0| Removed|
|DetailsList.Grouped.default.chromium.png|0| Removed|
|DetailsList.Grouped.hover.chromium.png|0| Removed|
|DetailsList.Root.click.chromium.png|0| Removed|
|DetailsList.Root.default.chromium.png|0| Removed|
|DetailsList.Root.hover.chromium.png|0| Removed|
|DetailsList.Single Selection Mode.click.chromium.png|0| Removed|
|DetailsList.Single Selection Mode.default.chromium.png|0| Removed|
|DetailsList.Single Selection Mode.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>Dialog</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Blocking.chromium.png|0| Added|
|Dialog.Large header.chromium.png|0| Added|
|Dialog.Root RTL.chromium.png|0| Added|
|Dialog.Root.chromium.png|0| Added|
|Dialog.Wide Dialog.chromium.png|0| Added|
|Dialog.Blocking.default.chromium.png|0| Removed|
|Dialog.Large header.default.chromium.png|0| Removed|
|Dialog.Root - RTL.default.chromium.png|0| Removed|
|Dialog.Root.default.chromium.png|0| Removed|
|Dialog.Wide Dialog.default.chromium.png|0| Removed|

</details>


  <details><summary><b>DocumentCard</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DocumentCard.Compact with preview image.chromium.png|0| Added|
|DocumentCard.Compact with preview list.chromium.png|0| Added|
|DocumentCard.Not truncated.chromium.png|0| Added|
|DocumentCard.With secondary title style.chromium.png|0| Added|
|DocumentCard.Compact with preview image.default.chromium.png|0| Removed|
|DocumentCard.Compact with preview list.default.chromium.png|0| Removed|
|DocumentCard.Not truncated.default.chromium.png|0| Removed|
|DocumentCard.With secondary title style.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Dropdown</b> 45 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.Custom Dropdown RTL.chromium.png|0| Added|
|Dropdown.Custom Dropdown.chromium.png|0| Added|
|Dropdown.Disabled option selected.chromium.png|0| Added|
|Dropdown.Multiselect RTL.chromium.png|0| Added|
|Dropdown.Multiselect.chromium.png|0| Added|
|Dropdown.Required without label.chromium.png|0| Added|
|Dropdown.Required.chromium.png|0| Added|
|Dropdown.Root RTL.chromium.png|0| Added|
|Dropdown.Root.chromium.png|0| Added|
|Dropdown.Custom Dropdown - RTL.click.chromium.png|0| Removed|
|Dropdown.Custom Dropdown - RTL.default.chromium.png|0| Removed|
|Dropdown.Custom Dropdown - RTL.hover item.chromium.png|0| Removed|
|Dropdown.Custom Dropdown - RTL.hover.chromium.png|0| Removed|
|Dropdown.Custom Dropdown.click.chromium.png|0| Removed|
|Dropdown.Custom Dropdown.default.chromium.png|0| Removed|
|Dropdown.Custom Dropdown.hover item.chromium.png|0| Removed|
|Dropdown.Custom Dropdown.hover.chromium.png|0| Removed|
|Dropdown.Disabled option selected.click.chromium.png|0| Removed|
|Dropdown.Disabled option selected.default.chromium.png|0| Removed|
|Dropdown.Disabled option selected.hover item.chromium.png|0| Removed|
|Dropdown.Disabled option selected.hover.chromium.png|0| Removed|
|Dropdown.Multiselect - RTL.click.chromium.png|0| Removed|
|Dropdown.Multiselect - RTL.default.chromium.png|0| Removed|
|Dropdown.Multiselect - RTL.hover item.chromium.png|0| Removed|
|Dropdown.Multiselect - RTL.hover.chromium.png|0| Removed|
|Dropdown.Multiselect.click.chromium.png|0| Removed|
|Dropdown.Multiselect.default.chromium.png|0| Removed|
|Dropdown.Multiselect.hover item.chromium.png|0| Removed|
|Dropdown.Multiselect.hover.chromium.png|0| Removed|
|Dropdown.Required without label.click.chromium.png|0| Removed|
|Dropdown.Required without label.default.chromium.png|0| Removed|
|Dropdown.Required without label.hover item.chromium.png|0| Removed|
|Dropdown.Required without label.hover.chromium.png|0| Removed|
|Dropdown.Required.click.chromium.png|0| Removed|
|Dropdown.Required.default.chromium.png|0| Removed|
|Dropdown.Required.hover item.chromium.png|0| Removed|
|Dropdown.Required.hover.chromium.png|0| Removed|
|Dropdown.Root - RTL.click.chromium.png|0| Removed|
|Dropdown.Root - RTL.default.chromium.png|0| Removed|
|Dropdown.Root - RTL.hover item.chromium.png|0| Removed|
|Dropdown.Root - RTL.hover.chromium.png|0| Removed|
|Dropdown.Root.click.chromium.png|0| Removed|
|Dropdown.Root.default.chromium.png|0| Removed|
|Dropdown.Root.hover item.chromium.png|0| Removed|
|Dropdown.Root.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>Dropdown Disabled</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown Disabled.Root.chromium.png|0| Added|
|Dropdown Disabled.Root.click.chromium.png|0| Removed|
|Dropdown Disabled.Root.default.chromium.png|0| Removed|
|Dropdown Disabled.Root.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>Facepile</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Facepile.Add face.chromium.png|0| Added|
|Facepile.Custom button styles.chromium.png|0| Added|
|Facepile.Extra extra small.chromium.png|0| Added|
|Facepile.Overflow.chromium.png|0| Added|
|Facepile.Root RTL.chromium.png|0| Added|
|Facepile.Root.chromium.png|0| Added|
|Facepile.Add face.default.chromium.png|0| Removed|
|Facepile.Custom button styles.default.chromium.png|0| Removed|
|Facepile.Extra extra small.default.chromium.png|0| Removed|
|Facepile.Overflow.default.chromium.png|0| Removed|
|Facepile.Root - RTL.default.chromium.png|0| Removed|
|Facepile.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>FocusTrapZones</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FocusTrapZones.Dialog Nested In Panel RTL.chromium.png|0| Added|
|FocusTrapZones.Dialog nested in Panel.chromium.png|0| Added|
|FocusTrapZones.Panel on its own.chromium.png|0| Added|
|FocusTrapZones.Dialog nested in Panel - RTL.click on panel close button.chromium.png|0| Removed|
|FocusTrapZones.Dialog nested in Panel - RTL.default.chromium.png|0| Removed|
|FocusTrapZones.Dialog nested in Panel.click on panel close button.chromium.png|0| Removed|
|FocusTrapZones.Dialog nested in Panel.default.chromium.png|0| Removed|
|FocusTrapZones.Panel on its own.click on panel close button.chromium.png|0| Removed|
|FocusTrapZones.Panel on its own.default.chromium.png|0| Removed|

</details>


  <details><summary><b>FolderCover</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FolderCover.Large Default Cover.chromium.png|0| Added|
|FolderCover.Large Media Cover.chromium.png|0| Added|
|FolderCover.Shared Cover.chromium.png|0| Added|
|FolderCover.Small Default Cover.chromium.png|0| Added|
|FolderCover.Small Media Cover.chromium.png|0| Added|
|FolderCover.Large Default Cover.default.chromium.png|0| Removed|
|FolderCover.Large Media Cover.default.chromium.png|0| Removed|
|FolderCover.Shared Cover.default.chromium.png|0| Removed|
|FolderCover.Small Default Cover.default.chromium.png|0| Removed|
|FolderCover.Small Media Cover.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Fonts</b> 32 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Fonts.Arabic.chromium.png|0| Added|
|Fonts.Armenian.chromium.png|0| Added|
|Fonts.Chinese (Simplified).chromium.png|0| Added|
|Fonts.Chinese (Traditional).chromium.png|0| Added|
|Fonts.Cyrillic.chromium.png|0| Added|
|Fonts.East European.chromium.png|0| Added|
|Fonts.Georgian.chromium.png|0| Added|
|Fonts.Greek.chromium.png|0| Added|
|Fonts.Hebrew.chromium.png|0| Added|
|Fonts.Hindi.chromium.png|0| Added|
|Fonts.Japanese.chromium.png|0| Added|
|Fonts.Korean.chromium.png|0| Added|
|Fonts.Thai.chromium.png|0| Added|
|Fonts.Vietnamese.chromium.png|0| Added|
|Fonts.Weights.chromium.png|0| Added|
|Fonts.West European.chromium.png|0| Added|
|Fonts.Arabic.default.chromium.png|0| Removed|
|Fonts.Armenian.default.chromium.png|0| Removed|
|Fonts.Chinese (Simplified).default.chromium.png|0| Removed|
|Fonts.Chinese (Traditional).default.chromium.png|0| Removed|
|Fonts.Cyrillic.default.chromium.png|0| Removed|
|Fonts.East European.default.chromium.png|0| Removed|
|Fonts.Georgian.default.chromium.png|0| Removed|
|Fonts.Greek.default.chromium.png|0| Removed|
|Fonts.Hebrew.default.chromium.png|0| Removed|
|Fonts.Hindi.default.chromium.png|0| Removed|
|Fonts.Japanese.default.chromium.png|0| Removed|
|Fonts.Korean.default.chromium.png|0| Removed|
|Fonts.Thai.default.chromium.png|0| Removed|
|Fonts.Vietnamese.default.chromium.png|0| Removed|
|Fonts.Weights.default.chromium.png|0| Removed|
|Fonts.West European.default.chromium.png|0| Removed|

</details>


  <details><summary><b>GroupedList</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|GroupedList.Root RTL.chromium.png|0| Added|
|GroupedList.Root.chromium.png|0| Added|
|GroupedList.Root - RTL.click.chromium.png|0| Removed|
|GroupedList.Root - RTL.default.chromium.png|0| Removed|
|GroupedList.Root - RTL.hover.chromium.png|0| Removed|
|GroupedList.Root.click.chromium.png|0| Removed|
|GroupedList.Root.default.chromium.png|0| Removed|
|GroupedList.Root.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>HoverCard</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|HoverCard.Root RTL.chromium.png|0| Added|
|HoverCard.Root.chromium.png|0| Added|
|HoverCard.Root - RTL.default.chromium.png|0| Removed|
|HoverCard.Root - RTL.fully expanded with test content.chromium.png|0| Removed|
|HoverCard.Root.default.chromium.png|0| Removed|
|HoverCard.Root.fully expanded with test content.chromium.png|0| Removed|

</details>


  <details><summary><b>Icon</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Icon.Color.chromium.png|0| Added|
|Icon.Image.chromium.png|0| Added|
|Icon.Root.chromium.png|0| Added|
|Icon.Color.default.chromium.png|0| Removed|
|Icon.Image.default.chromium.png|0| Removed|
|Icon.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>IconButton Scenarios (compat)</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|IconButton Scenarios (compat).icon button with menu.chromium.png|0| Added|
|IconButton Scenarios (compat).normal icon button.chromium.png|0| Added|
|IconButton Scenarios (compat).icon button with menu.hover icon.chromium.png|0| Removed|
|IconButton Scenarios (compat).icon button with menu.icon.chromium.png|0| Removed|
|IconButton Scenarios (compat).normal icon button.hover icon.chromium.png|0| Removed|
|IconButton Scenarios (compat).normal icon button.icon.chromium.png|0| Removed|

</details>


  <details><summary><b>Image</b> 42 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- CenterContain, image larger.chromium.png|0| Added|
|Image.Fit- CenterContain, image smaller.chromium.png|0| Added|
|Image.Fit- CenterContain, image taller.chromium.png|0| Added|
|Image.Fit- CenterContain, image wider.chromium.png|0| Added|
|Image.Fit- center, image larger.chromium.png|0| Added|
|Image.Fit- center, image smaller.chromium.png|0| Added|
|Image.Fit- centerCover, image larger.chromium.png|0| Added|
|Image.Fit- centerCover, image smaller.chromium.png|0| Added|
|Image.Fit- centerCover, image taller.chromium.png|0| Added|
|Image.Fit- centerCover, image wider.chromium.png|0| Added|
|Image.Fit- contain, image taller.chromium.png|0| Added|
|Image.Fit- contain, image wider.chromium.png|0| Added|
|Image.Fit- cover, image taller.chromium.png|0| Added|
|Image.Fit- cover, image wider.chromium.png|0| Added|
|Image.Fit- none, image larger.chromium.png|0| Added|
|Image.Fit- none, image smaller.chromium.png|0| Added|
|Image.Maximize frame, landscape container.chromium.png|0| Added|
|Image.Maximize frame, portrait container.chromium.png|0| Added|
|Image.No fit, no w-h.chromium.png|0| Added|
|Image.No fit, only height.chromium.png|0| Added|
|Image.No fit, only width.chromium.png|0| Added|
|Image.Fit- CenterContain, image larger.default.chromium.png|0| Removed|
|Image.Fit- CenterContain, image smaller.default.chromium.png|0| Removed|
|Image.Fit- CenterContain, image taller.default.chromium.png|0| Removed|
|Image.Fit- CenterContain, image wider.default.chromium.png|0| Removed|
|Image.Fit- center, image larger.default.chromium.png|0| Removed|
|Image.Fit- center, image smaller.default.chromium.png|0| Removed|
|Image.Fit- centerCover, image larger.default.chromium.png|0| Removed|
|Image.Fit- centerCover, image smaller.default.chromium.png|0| Removed|
|Image.Fit- centerCover, image taller.default.chromium.png|0| Removed|
|Image.Fit- centerCover, image wider.default.chromium.png|0| Removed|
|Image.Fit- contain, image taller.default.chromium.png|0| Removed|
|Image.Fit- contain, image wider.default.chromium.png|0| Removed|
|Image.Fit- cover, image taller.default.chromium.png|0| Removed|
|Image.Fit- cover, image wider.default.chromium.png|0| Removed|
|Image.Fit- none, image larger.default.chromium.png|0| Removed|
|Image.Fit- none, image smaller.default.chromium.png|0| Removed|
|Image.Maximize frame, landscape container.default.chromium.png|0| Removed|
|Image.Maximize frame, portrait container.default.chromium.png|0| Removed|
|Image.No fit, no w-h.default.chromium.png|0| Removed|
|Image.No fit, only height.default.chromium.png|0| Removed|
|Image.No fit, only width.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Keytip</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Disabled.chromium.png|0| Added|
|Keytip.Offset.chromium.png|0| Added|
|Keytip.Root.chromium.png|0| Added|
|Keytip.Disabled.default.chromium.png|0| Removed|
|Keytip.Offset.default.chromium.png|0| Removed|
|Keytip.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Label</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.Disabled.chromium.png|0| Added|
|Label.Required.chromium.png|0| Added|
|Label.Root RTL.chromium.png|0| Added|
|Label.Root.chromium.png|0| Added|
|Label.Disabled.default.chromium.png|0| Removed|
|Label.Required.default.chromium.png|0| Removed|
|Label.Root - RTL.default.chromium.png|0| Removed|
|Label.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Layer</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Layer.Root RTL.chromium.png|0| Added|
|Layer.Root.chromium.png|0| Added|
|Layer.Root - RTL.default.chromium.png|0| Removed|
|Layer.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Link</b> 35 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link.Disabled.chromium.png|0| Added|
|Link.No Href Disabled.chromium.png|0| Added|
|Link.No Href.chromium.png|0| Added|
|Link.Root RTL.chromium.png|0| Added|
|Link.Root.chromium.png|0| Added|
|Link.Underlined Disabled.chromium.png|0| Added|
|Link.Underlined.chromium.png|0| Added|
|Link.Disabled.click.chromium.png|0| Removed|
|Link.Disabled.default.chromium.png|0| Removed|
|Link.Disabled.focus.chromium.png|0| Removed|
|Link.Disabled.hover.chromium.png|0| Removed|
|Link.No Href Disabled.click.chromium.png|0| Removed|
|Link.No Href Disabled.default.chromium.png|0| Removed|
|Link.No Href Disabled.focus.chromium.png|0| Removed|
|Link.No Href Disabled.hover.chromium.png|0| Removed|
|Link.No Href.click.chromium.png|0| Removed|
|Link.No Href.default.chromium.png|0| Removed|
|Link.No Href.focus.chromium.png|0| Removed|
|Link.No Href.hover.chromium.png|0| Removed|
|Link.Root - RTL.click.chromium.png|0| Removed|
|Link.Root - RTL.default.chromium.png|0| Removed|
|Link.Root - RTL.focus.chromium.png|0| Removed|
|Link.Root - RTL.hover.chromium.png|0| Removed|
|Link.Root.click.chromium.png|0| Removed|
|Link.Root.default.chromium.png|0| Removed|
|Link.Root.focus.chromium.png|0| Removed|
|Link.Root.hover.chromium.png|0| Removed|
|Link.Underlined Disabled.click.chromium.png|0| Removed|
|Link.Underlined Disabled.default.chromium.png|0| Removed|
|Link.Underlined Disabled.focus.chromium.png|0| Removed|
|Link.Underlined Disabled.hover.chromium.png|0| Removed|
|Link.Underlined.click.chromium.png|0| Removed|
|Link.Underlined.default.chromium.png|0| Removed|
|Link.Underlined.focus.chromium.png|0| Removed|
|Link.Underlined.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>List</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|List.Root RTL.chromium.png|0| Added|
|List.Root.chromium.png|0| Added|
|List.Root - RTL.default.chromium.png|0| Removed|
|List.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>MediaTile</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MediaTile.Media tile with nameplate hidden until hover.chromium.png|0| Added|
|MediaTile.Media tile with single activity line.chromium.png|0| Added|
|MediaTile.Media tile with two activity lines.chromium.png|0| Added|
|MediaTile.Media tile with very long name and activity.chromium.png|0| Added|
|MediaTile.Media tile with nameplate hidden until hover.default.chromium.png|0| Removed|
|MediaTile.Media tile with nameplate hidden until hover.hover.chromium.png|0| Removed|
|MediaTile.Media tile with single activity line.default.chromium.png|0| Removed|
|MediaTile.Media tile with single activity line.hover.chromium.png|0| Removed|
|MediaTile.Media tile with two activity lines.default.chromium.png|0| Removed|
|MediaTile.Media tile with two activity lines.hover.chromium.png|0| Removed|
|MediaTile.Media tile with very long name and activity.default.chromium.png|0| Removed|
|MediaTile.Media tile with very long name and activity.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>MessageBar</b> 48 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Blocked.chromium.png|0| Added|
|MessageBar.Error.chromium.png|0| Added|
|MessageBar.Root Actions RTL.chromium.png|0| Added|
|MessageBar.Root Actions Single Line RTL.chromium.png|0| Added|
|MessageBar.Root Dismiss And Action RTL.chromium.png|0| Added|
|MessageBar.Root Dismiss And Action Single Line RTL.chromium.png|0| Added|
|MessageBar.Root Dismiss RTL.chromium.png|0| Added|
|MessageBar.Root Multiline RTL.chromium.png|0| Added|
|MessageBar.Root Overflow RTL.chromium.png|0| Added|
|MessageBar.Root RTL.chromium.png|0| Added|
|MessageBar.Root Truncated RTL.chromium.png|0| Added|
|MessageBar.Root actions single line.chromium.png|0| Added|
|MessageBar.Root actions.chromium.png|0| Added|
|MessageBar.Root dismiss and action single line.chromium.png|0| Added|
|MessageBar.Root dismiss and action.chromium.png|0| Added|
|MessageBar.Root dismiss single line.chromium.png|0| Added|
|MessageBar.Root dismiss.chromium.png|0| Added|
|MessageBar.Root multiline.chromium.png|0| Added|
|MessageBar.Root overflow.chromium.png|0| Added|
|MessageBar.Root truncated.chromium.png|0| Added|
|MessageBar.Root.chromium.png|0| Added|
|MessageBar.Severe Warning.chromium.png|0| Added|
|MessageBar.Success.chromium.png|0| Added|
|MessageBar.Warning.chromium.png|0| Added|
|MessageBar.Blocked.default.chromium.png|0| Removed|
|MessageBar.Error.default.chromium.png|0| Removed|
|MessageBar.Root - RTL.default.chromium.png|0| Removed|
|MessageBar.Root actions - RTL.default.chromium.png|0| Removed|
|MessageBar.Root actions single line - RTL.default.chromium.png|0| Removed|
|MessageBar.Root actions single line.default.chromium.png|0| Removed|
|MessageBar.Root actions.default.chromium.png|0| Removed|
|MessageBar.Root dismiss - RTL.default.chromium.png|0| Removed|
|MessageBar.Root dismiss and action - RTL.default.chromium.png|0| Removed|
|MessageBar.Root dismiss and action single line - RTL.default.chromium.png|0| Removed|
|MessageBar.Root dismiss and action single line.default.chromium.png|0| Removed|
|MessageBar.Root dismiss and action.default.chromium.png|0| Removed|
|MessageBar.Root dismiss single line.default.chromium.png|0| Removed|
|MessageBar.Root dismiss.default.chromium.png|0| Removed|
|MessageBar.Root multiline - RTL.default.chromium.png|0| Removed|
|MessageBar.Root multiline.default.chromium.png|0| Removed|
|MessageBar.Root overflow - RTL.default.chromium.png|0| Removed|
|MessageBar.Root overflow.default.chromium.png|0| Removed|
|MessageBar.Root truncated - RTL.default.chromium.png|0| Removed|
|MessageBar.Root truncated.default.chromium.png|0| Removed|
|MessageBar.Root.default.chromium.png|0| Removed|
|MessageBar.Severe Warning.default.chromium.png|0| Removed|
|MessageBar.Success.default.chromium.png|0| Removed|
|MessageBar.Warning.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Modal</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Modal.Modeless.chromium.png|0| Added|
|Modal.Root.chromium.png|0| Added|
|Modal.Modeless.default.chromium.png|0| Removed|
|Modal.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Nav</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Nav.Disabled RTL.chromium.png|0| Added|
|Nav.Disabled.chromium.png|0| Added|
|Nav.Root RTL.chromium.png|0| Added|
|Nav.Root.chromium.png|0| Added|
|Nav.Disabled - RTL.click.chromium.png|0| Removed|
|Nav.Disabled - RTL.default.chromium.png|0| Removed|
|Nav.Disabled - RTL.hover.chromium.png|0| Removed|
|Nav.Disabled.click.chromium.png|0| Removed|
|Nav.Disabled.default.chromium.png|0| Removed|
|Nav.Disabled.hover.chromium.png|0| Removed|
|Nav.Root - RTL.click.chromium.png|0| Removed|
|Nav.Root - RTL.default.chromium.png|0| Removed|
|Nav.Root - RTL.hover.chromium.png|0| Removed|
|Nav.Root.click.chromium.png|0| Removed|
|Nav.Root.default.chromium.png|0| Removed|
|Nav.Root.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>OverflowSet</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet.Root RTL.chromium.png|0| Added|
|OverflowSet.Root.chromium.png|0| Added|
|OverflowSet.Root - RTL.default.chromium.png|0| Removed|
|OverflowSet.Root - RTL.default.chromium_1.png|0| Removed|
|OverflowSet.Root.default.chromium.png|0| Removed|
|OverflowSet.Root.default.chromium_1.png|0| Removed|

</details>


  <details><summary><b>OverflowSet variant</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet variant.Vertical Direction.chromium.png|0| Added|
|OverflowSet variant.Vertical Direction.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Overlay</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Overlay.Dark.chromium.png|0| Added|
|Overlay.Root RTL.chromium.png|0| Added|
|Overlay.Root.chromium.png|0| Added|
|Overlay.Dark.default.chromium.png|0| Removed|
|Overlay.Root - RTL.default.chromium.png|0| Removed|
|Overlay.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Panel</b> 38 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Panel.Custom anchored left.chromium.png|0| Added|
|Panel.Custom.chromium.png|0| Added|
|Panel.Extra large right.chromium.png|0| Added|
|Panel.Large fixed right.chromium.png|0| Added|
|Panel.Large right.chromium.png|0| Added|
|Panel.Medium Right RTL.chromium.png|0| Added|
|Panel.Medium right.chromium.png|0| Added|
|Panel.Search Box And Right Panel RTL.chromium.png|0| Added|
|Panel.SearchBox and Right Panel.chromium.png|0| Added|
|Panel.Small Fixed Right W Close Button RTL.chromium.png|0| Added|
|Panel.Small Left W Close Button RTL.chromium.png|0| Added|
|Panel.Small fixed right w- close button.chromium.png|0| Added|
|Panel.Small fluid right.chromium.png|0| Added|
|Panel.Small left w- close button.chromium.png|0| Added|
|Panel.With custom navigation.chromium.png|0| Added|
|Panel.With footer at the bottom.chromium.png|0| Added|
|Panel.With no header, close button.chromium.png|0| Added|
|Panel.With no navigation.chromium.png|0| Added|
|Panel.Custom anchored left.default.chromium.png|0| Removed|
|Panel.Custom.default.chromium.png|0| Removed|
|Panel.Extra large right.default.chromium.png|0| Removed|
|Panel.Large fixed right.default.chromium.png|0| Removed|
|Panel.Large right.default.chromium.png|0| Removed|
|Panel.Medium right - RTL.default.chromium.png|0| Removed|
|Panel.Medium right.default.chromium.png|0| Removed|
|Panel.SearchBox and Right Panel - RTL.click.chromium.png|0| Removed|
|Panel.SearchBox and Right Panel - RTL.default.chromium.png|0| Removed|
|Panel.SearchBox and Right Panel.click.chromium.png|0| Removed|
|Panel.SearchBox and Right Panel.default.chromium.png|0| Removed|
|Panel.Small fixed right w- close button - RTL.default.chromium.png|0| Removed|
|Panel.Small fixed right w- close button.default.chromium.png|0| Removed|
|Panel.Small fluid right.default.chromium.png|0| Removed|
|Panel.Small left w- close button - RTL.default.chromium.png|0| Removed|
|Panel.Small left w- close button.default.chromium.png|0| Removed|
|Panel.With custom navigation.default.chromium.png|0| Removed|
|Panel.With footer at the bottom.default.chromium.png|0| Removed|
|Panel.With no header, close button.default.chromium.png|0| Removed|
|Panel.With no navigation.default.chromium.png|0| Removed|

</details>


  <details><summary><b>PeoplePicker</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Compact Selected RTL.chromium.png|0| Added|
|PeoplePicker.Compact selected.chromium.png|0| Added|
|PeoplePicker.Compact.chromium.png|0| Added|
|PeoplePicker.List Selected RTL.chromium.png|0| Added|
|PeoplePicker.List selected.chromium.png|0| Added|
|PeoplePicker.List.chromium.png|0| Added|
|PeoplePicker.Normal Overflow Selected RTL.chromium.png|0| Added|
|PeoplePicker.Normal Overflow selected.chromium.png|0| Added|
|PeoplePicker.Normal Selected RTL.chromium.png|0| Added|
|PeoplePicker.Normal selected.chromium.png|0| Added|
|PeoplePicker.Normal with text.chromium.png|0| Added|
|PeoplePicker.Normal.chromium.png|0| Added|
|PeoplePicker.Compact selected - RTL.default.chromium.png|0| Removed|
|PeoplePicker.Compact selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.Compact selected.default.chromium.png|0| Removed|
|PeoplePicker.Compact selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.Compact.default.chromium.png|0| Removed|
|PeoplePicker.Compact.suggestions.chromium.png|0| Removed|
|PeoplePicker.List selected - RTL.default.chromium.png|0| Removed|
|PeoplePicker.List selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.List selected.default.chromium.png|0| Removed|
|PeoplePicker.List selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.List.default.chromium.png|0| Removed|
|PeoplePicker.List.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal Overflow selected - RTL.default.chromium.png|0| Removed|
|PeoplePicker.Normal Overflow selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal Overflow selected.default.chromium.png|0| Removed|
|PeoplePicker.Normal Overflow selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal selected - RTL.default.chromium.png|0| Removed|
|PeoplePicker.Normal selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal selected.default.chromium.png|0| Removed|
|PeoplePicker.Normal selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal with text.default.chromium.png|0| Removed|
|PeoplePicker.Normal with text.suggestion-  a .chromium.png|0| Removed|
|PeoplePicker.Normal.default.chromium.png|0| Removed|
|PeoplePicker.Normal.suggestions.chromium.png|0| Removed|

</details>


  <details><summary><b>Persona</b> 32 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona.Initials RTL.chromium.png|0| Added|
|Persona.Initials.chromium.png|0| Added|
|Persona.Persona with children.chromium.png|0| Added|
|Persona.Size 100 Extra Large RTL.chromium.png|0| Added|
|Persona.Size 120 RTL.chromium.png|0| Added|
|Persona.default size, details hidden.chromium.png|0| Added|
|Persona.default size, presences.chromium.png|0| Added|
|Persona.size10 (tiny).chromium.png|0| Added|
|Persona.size100 (extraLarge).chromium.png|0| Added|
|Persona.size120.chromium.png|0| Added|
|Persona.size24 (extraExtraSmall).chromium.png|0| Added|
|Persona.size28 (extraSmall).chromium.png|0| Added|
|Persona.size32.chromium.png|0| Added|
|Persona.size40 (small).chromium.png|0| Added|
|Persona.size48 (regular).chromium.png|0| Added|
|Persona.size72 (large).chromium.png|0| Added|
|Persona.Initials - RTL.default.chromium.png|0| Removed|
|Persona.Initials.default.chromium.png|0| Removed|
|Persona.Persona with children.default.chromium.png|0| Removed|
|Persona.default size, details hidden.default.chromium.png|0| Removed|
|Persona.default size, presences.default.chromium.png|0| Removed|
|Persona.size10 (tiny).default.chromium.png|0| Removed|
|Persona.size100 (extraLarge) - RTL.default.chromium.png|0| Removed|
|Persona.size100 (extraLarge).default.chromium.png|0| Removed|
|Persona.size120 - RTL.default.chromium.png|0| Removed|
|Persona.size120.default.chromium.png|0| Removed|
|Persona.size24 (extraExtraSmall).default.chromium.png|0| Removed|
|Persona.size28 (extraSmall).default.chromium.png|0| Removed|
|Persona.size32.default.chromium.png|0| Removed|
|Persona.size40 (small).default.chromium.png|0| Removed|
|Persona.size48 (regular).default.chromium.png|0| Removed|
|Persona.size72 (large).default.chromium.png|0| Removed|

</details>


  <details><summary><b>Pivot</b> 35 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot.Count And Icon RTL.chromium.png|0| Added|
|Pivot.Count and icon.chromium.png|0| Added|
|Pivot.Large.chromium.png|0| Added|
|Pivot.Root.chromium.png|0| Added|
|Pivot.Tabs RTL.chromium.png|0| Added|
|Pivot.Tabs large.chromium.png|0| Added|
|Pivot.Tabs.chromium.png|0| Added|
|Pivot.Count and icon - RTL.click.chromium.png|0| Removed|
|Pivot.Count and icon - RTL.default.chromium.png|0| Removed|
|Pivot.Count and icon - RTL.hover-selected.chromium.png|0| Removed|
|Pivot.Count and icon - RTL.hover.chromium.png|0| Removed|
|Pivot.Count and icon.click.chromium.png|0| Removed|
|Pivot.Count and icon.default.chromium.png|0| Removed|
|Pivot.Count and icon.hover-selected.chromium.png|0| Removed|
|Pivot.Count and icon.hover.chromium.png|0| Removed|
|Pivot.Large.click.chromium.png|0| Removed|
|Pivot.Large.default.chromium.png|0| Removed|
|Pivot.Large.hover-selected.chromium.png|0| Removed|
|Pivot.Large.hover.chromium.png|0| Removed|
|Pivot.Root.click.chromium.png|0| Removed|
|Pivot.Root.default.chromium.png|0| Removed|
|Pivot.Root.hover-selected.chromium.png|0| Removed|
|Pivot.Root.hover.chromium.png|0| Removed|
|Pivot.Tabs - RTL.click.chromium.png|0| Removed|
|Pivot.Tabs - RTL.default.chromium.png|0| Removed|
|Pivot.Tabs - RTL.hover-selected.chromium.png|0| Removed|
|Pivot.Tabs - RTL.hover.chromium.png|0| Removed|
|Pivot.Tabs large.click.chromium.png|0| Removed|
|Pivot.Tabs large.default.chromium.png|0| Removed|
|Pivot.Tabs large.hover-selected.chromium.png|0| Removed|
|Pivot.Tabs large.hover.chromium.png|0| Removed|
|Pivot.Tabs.click.chromium.png|0| Removed|
|Pivot.Tabs.default.chromium.png|0| Removed|
|Pivot.Tabs.hover-selected.chromium.png|0| Removed|
|Pivot.Tabs.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>ProgressIndicator</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressIndicator.0%.chromium.png|0| Added|
|ProgressIndicator.100%.chromium.png|0| Added|
|ProgressIndicator.50 RTL.chromium.png|0| Added|
|ProgressIndicator.50%.chromium.png|0| Added|
|ProgressIndicator.0%.default.chromium.png|0| Removed|
|ProgressIndicator.100%.default.chromium.png|0| Removed|
|ProgressIndicator.50% - RTL.default.chromium.png|0| Removed|
|ProgressIndicator.50%.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Rating</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Rating.Disabled.chromium.png|0| Added|
|Rating.Large.chromium.png|0| Added|
|Rating.Rated RTL.chromium.png|0| Added|
|Rating.Rated.chromium.png|0| Added|
|Rating.Root.chromium.png|0| Added|
|Rating.Disabled.click.chromium.png|0| Removed|
|Rating.Disabled.default.chromium.png|0| Removed|
|Rating.Large.click.chromium.png|0| Removed|
|Rating.Large.default.chromium.png|0| Removed|
|Rating.Rated - RTL.click.chromium.png|0| Removed|
|Rating.Rated - RTL.default.chromium.png|0| Removed|
|Rating.Rated.click.chromium.png|0| Removed|
|Rating.Rated.default.chromium.png|0| Removed|
|Rating.Root.click.chromium.png|0| Removed|
|Rating.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>ResizeGroup</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ResizeGroup.Root RTL.chromium.png|0| Added|
|ResizeGroup.Root.chromium.png|0| Added|
|ResizeGroup.Root - RTL.click overflow.chromium.png|0| Removed|
|ResizeGroup.Root - RTL.default.chromium.png|0| Removed|
|ResizeGroup.Root.click overflow.chromium.png|0| Removed|
|ResizeGroup.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>ScrollablePane</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ScrollablePane.Default ScrollablePane Example.chromium.png|0| Added|
|ScrollablePane.Default ScrollablePane Example.default.chromium.png|0| Removed|
|ScrollablePane.Default ScrollablePane Example.scrolled.chromium.png|0| Removed|

</details>


  <details><summary><b>ScrollablePane Details List</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.chromium.png|0| Added|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.default.chromium.png|0| Removed|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll down by a small amount so that the first row is still visible.chromium.png|0| Removed|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll down to the bottom.chromium.png|0| Removed|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll up to the top.chromium.png|0| Removed|

</details>


  <details><summary><b>ScrollablePane Grouped Details List</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.chromium.png|0| Added|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.If groups are expanded, on horizontal scroll, over scroll should not happen for content container.chromium.png|0| Removed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.If groups are expanded, when header becomes sticky, horizontal scrollbar should be visible.chromium.png|0| Removed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.On expanding a group, horizontal scrollbar should be visible.chromium.png|0| Removed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling down to the bottom with non-zero scrollLeft.chromium.png|0| Removed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling down to the bottom.chromium.png|0| Removed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling up to the top with non-zero scrollLeft.chromium.png|0| Removed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling up to the top.chromium.png|0| Removed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility when header is sticky and scrollLeft is non-zero.chromium.png|0| Removed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility when header is sticky.chromium.png|0| Removed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibilty after scrolling left.chromium.png|0| Removed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.default- scrollbars should be visible.chromium.png|0| Removed|

</details>


  <details><summary><b>SearchBox</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox.Full RTL.chromium.png|0| Added|
|SearchBox.Full.chromium.png|0| Added|
|SearchBox.Root RTL.chromium.png|0| Added|
|SearchBox.Root.chromium.png|0| Added|
|SearchBox.Show Icon RTL.chromium.png|0| Added|
|SearchBox.ShowIcon.chromium.png|0| Added|
|SearchBox.Full - RTL.default.chromium.png|0| Removed|
|SearchBox.Full - RTL.default.chromium_1.png|0| Removed|
|SearchBox.Full.default.chromium.png|0| Removed|
|SearchBox.Full.default.chromium_1.png|0| Removed|
|SearchBox.Root - RTL.default.chromium.png|0| Removed|
|SearchBox.Root - RTL.default.chromium_1.png|0| Removed|
|SearchBox.Root.default.chromium.png|0| Removed|
|SearchBox.Root.default.chromium_1.png|0| Removed|
|SearchBox.ShowIcon - RTL.default.chromium.png|0| Removed|
|SearchBox.ShowIcon - RTL.default.chromium_1.png|0| Removed|
|SearchBox.ShowIcon.default.chromium.png|0| Removed|
|SearchBox.ShowIcon.default.chromium_1.png|0| Removed|

</details>


  <details><summary><b>Separator</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Separator.Horizontal End Aligned RTL.chromium.png|0| Added|
|Separator.Horizontal End Aligned.chromium.png|0| Added|
|Separator.Horizontal Start Aligned RTL.chromium.png|0| Added|
|Separator.Horizontal Start Aligned.chromium.png|0| Added|
|Separator.Root RTL.chromium.png|0| Added|
|Separator.Root.chromium.png|0| Added|
|Separator.Vertical Center Aligned RTL.chromium.png|0| Added|
|Separator.Vertical Center Aligned.chromium.png|0| Added|
|Separator.Vertical End Aligned RTL.chromium.png|0| Added|
|Separator.Vertical End Aligned.chromium.png|0| Added|
|Separator.Vertical Start Aligned RTL.chromium.png|0| Added|
|Separator.Vertical Start Aligned.chromium.png|0| Added|
|Separator.Horizontal End Aligned - RTL.default.chromium.png|0| Removed|
|Separator.Horizontal End Aligned.default.chromium.png|0| Removed|
|Separator.Horizontal Start Aligned - RTL.default.chromium.png|0| Removed|
|Separator.Horizontal Start Aligned.default.chromium.png|0| Removed|
|Separator.Root - RTL.default.chromium.png|0| Removed|
|Separator.Root.default.chromium.png|0| Removed|
|Separator.Vertical Center Aligned - RTL.default.chromium.png|0| Removed|
|Separator.Vertical Center Aligned.default.chromium.png|0| Removed|
|Separator.Vertical End Aligned - RTL.default.chromium.png|0| Removed|
|Separator.Vertical End Aligned.default.chromium.png|0| Removed|
|Separator.Vertical Start Aligned - RTL.default.chromium.png|0| Removed|
|Separator.Vertical Start Aligned.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Shimmer</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shimmer.50% width.chromium.png|0| Added|
|Shimmer.Basic.chromium.png|0| Added|
|Shimmer.Circle Gap Line RTL.chromium.png|0| Added|
|Shimmer.Circle Gap Line.chromium.png|0| Added|
|Shimmer.Custom elements on themed background.chromium.png|0| Added|
|Shimmer.Custom elements.chromium.png|0| Added|
|Shimmer.Data loaded.chromium.png|0| Added|
|Shimmer.Data not loaded.chromium.png|0| Added|
|Shimmer.50% width.default.chromium.png|0| Removed|
|Shimmer.Basic.default.chromium.png|0| Removed|
|Shimmer.Circle Gap Line - RTL.default.chromium.png|0| Removed|
|Shimmer.Circle Gap Line.default.chromium.png|0| Removed|
|Shimmer.Custom elements on themed background.default.chromium.png|0| Removed|
|Shimmer.Custom elements.default.chromium.png|0| Removed|
|Shimmer.Data loaded.default.chromium.png|0| Removed|
|Shimmer.Data not loaded.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Signals</b> 42 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Signals.Awaiting approval.chromium.png|0| Added|
|Signals.Blocked.chromium.png|0| Added|
|Signals.Comments (count).chromium.png|0| Added|
|Signals.Comments.chromium.png|0| Added|
|Signals.Emailed.chromium.png|0| Added|
|Signals.Follow.chromium.png|0| Added|
|Signals.Malware detected.chromium.png|0| Added|
|Signals.Mention.chromium.png|0| Added|
|Signals.Missing metadata.chromium.png|0| Added|
|Signals.New (positioning).chromium.png|0| Added|
|Signals.New.chromium.png|0| Added|
|Signals.NotFollow.chromium.png|0| Added|
|Signals.Read-only.chromium.png|0| Added|
|Signals.Record.chromium.png|0| Added|
|Signals.Shared.chromium.png|0| Added|
|Signals.Someone checked out.chromium.png|0| Added|
|Signals.Trending.chromium.png|0| Added|
|Signals.Unseen edit.chromium.png|0| Added|
|Signals.Unseen reply.chromium.png|0| Added|
|Signals.Warning.chromium.png|0| Added|
|Signals.You checked out.chromium.png|0| Added|
|Signals.Awaiting approval.default.chromium.png|0| Removed|
|Signals.Blocked.default.chromium.png|0| Removed|
|Signals.Comments (count).default.chromium.png|0| Removed|
|Signals.Comments.default.chromium.png|0| Removed|
|Signals.Emailed.default.chromium.png|0| Removed|
|Signals.Follow.default.chromium.png|0| Removed|
|Signals.Malware detected.default.chromium.png|0| Removed|
|Signals.Mention.default.chromium.png|0| Removed|
|Signals.Missing metadata.default.chromium.png|0| Removed|
|Signals.New (positioning).default.chromium.png|0| Removed|
|Signals.New.default.chromium.png|0| Removed|
|Signals.NotFollow.default.chromium.png|0| Removed|
|Signals.Read-only.default.chromium.png|0| Removed|
|Signals.Record.default.chromium.png|0| Removed|
|Signals.Shared.default.chromium.png|0| Removed|
|Signals.Someone checked out.default.chromium.png|0| Removed|
|Signals.Trending.default.chromium.png|0| Removed|
|Signals.Unseen edit.default.chromium.png|0| Removed|
|Signals.Unseen reply.default.chromium.png|0| Removed|
|Signals.Warning.default.chromium.png|0| Removed|
|Signals.You checked out.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Slider</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Disabled.chromium.png|0| Added|
|Slider.EqualMinMax.chromium.png|0| Added|
|Slider.Max not multiple of step.chromium.png|0| Added|
|Slider.Root RTL.chromium.png|0| Added|
|Slider.Root.chromium.png|0| Added|
|Slider.Step less than 1.chromium.png|0| Added|
|Slider.Vertical.chromium.png|0| Added|
|Slider.Disabled.default.chromium.png|0| Removed|
|Slider.Disabled.hover.chromium.png|0| Removed|
|Slider.EqualMinMax.default.chromium.png|0| Removed|
|Slider.EqualMinMax.hover.chromium.png|0| Removed|
|Slider.Max not multiple of step.default.chromium.png|0| Removed|
|Slider.Max not multiple of step.hover.chromium.png|0| Removed|
|Slider.Root - RTL.default.chromium.png|0| Removed|
|Slider.Root - RTL.hover.chromium.png|0| Removed|
|Slider.Root.default.chromium.png|0| Removed|
|Slider.Root.hover.chromium.png|0| Removed|
|Slider.Step less than 1.default.chromium.png|0| Removed|
|Slider.Step less than 1.hover.chromium.png|0| Removed|
|Slider.Vertical.default.chromium.png|0| Removed|
|Slider.Vertical.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>SpinButton</b> 40 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton.Disabled.chromium.png|0| Added|
|SpinButton.Label At End RTL.chromium.png|0| Added|
|SpinButton.Label At End With Icon RTL.chromium.png|0| Added|
|SpinButton.Label at end with icon.chromium.png|0| Added|
|SpinButton.Label at end.chromium.png|0| Added|
|SpinButton.Label on bottom with icon.chromium.png|0| Added|
|SpinButton.Label on bottom.chromium.png|0| Added|
|SpinButton.Label on top with icon.chromium.png|0| Added|
|SpinButton.Label on top.chromium.png|0| Added|
|SpinButton.Root.chromium.png|0| Added|
|SpinButton.With Icon RTL.chromium.png|0| Added|
|SpinButton.With icon.chromium.png|0| Added|
|SpinButton.Disabled.click input.chromium.png|0| Removed|
|SpinButton.Disabled.default.chromium.png|0| Removed|
|SpinButton.Disabled.hover arrow.chromium.png|0| Removed|
|SpinButton.Disabled.hover input.chromium.png|0| Removed|
|SpinButton.Disabled.mouseDown arrow.chromium.png|0| Removed|
|SpinButton.Label at end - RTL.default.chromium.png|0| Removed|
|SpinButton.Label at end with icon - RTL.default.chromium.png|0| Removed|
|SpinButton.Label at end with icon.default.chromium.png|0| Removed|
|SpinButton.Label at end.default.chromium.png|0| Removed|
|SpinButton.Label on bottom with icon.default.chromium.png|0| Removed|
|SpinButton.Label on bottom.default.chromium.png|0| Removed|
|SpinButton.Label on top with icon.default.chromium.png|0| Removed|
|SpinButton.Label on top.default.chromium.png|0| Removed|
|SpinButton.Root.click input.chromium.png|0| Removed|
|SpinButton.Root.default.chromium.png|0| Removed|
|SpinButton.Root.hover arrow.chromium.png|0| Removed|
|SpinButton.Root.hover input.chromium.png|0| Removed|
|SpinButton.Root.mouseDown arrow.chromium.png|0| Removed|
|SpinButton.With icon - RTL.click input.chromium.png|0| Removed|
|SpinButton.With icon - RTL.default.chromium.png|0| Removed|
|SpinButton.With icon - RTL.hover arrow.chromium.png|0| Removed|
|SpinButton.With icon - RTL.hover input.chromium.png|0| Removed|
|SpinButton.With icon - RTL.mouseDown arrow.chromium.png|0| Removed|
|SpinButton.With icon.click input.chromium.png|0| Removed|
|SpinButton.With icon.default.chromium.png|0| Removed|
|SpinButton.With icon.hover arrow.chromium.png|0| Removed|
|SpinButton.With icon.hover input.chromium.png|0| Removed|
|SpinButton.With icon.mouseDown arrow.chromium.png|0| Removed|

</details>


  <details><summary><b>Spinner</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner.Extra small.chromium.png|0| Added|
|Spinner.Label at top.chromium.png|0| Added|
|Spinner.Label on the left.chromium.png|0| Added|
|Spinner.Label on the right.chromium.png|0| Added|
|Spinner.Label.chromium.png|0| Added|
|Spinner.Large.chromium.png|0| Added|
|Spinner.Medium.chromium.png|0| Added|
|Spinner.Small.chromium.png|0| Added|
|Spinner.Extra small.default.chromium.png|0| Removed|
|Spinner.Label at top.default.chromium.png|0| Removed|
|Spinner.Label on the left.default.chromium.png|0| Removed|
|Spinner.Label on the right.default.chromium.png|0| Removed|
|Spinner.Label.default.chromium.png|0| Removed|
|Spinner.Large.default.chromium.png|0| Removed|
|Spinner.Medium.default.chromium.png|0| Removed|
|Spinner.Small.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Stack</b> 88 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Stack.Horizontal Stack - Bottom-aligned.chromium.png|0| Added|
|Stack.Horizontal Stack - Box shadow around items.chromium.png|0| Added|
|Stack.Horizontal Stack - Default.chromium.png|0| Added|
|Stack.Horizontal Stack - Gap.chromium.png|0| Added|
|Stack.Horizontal Stack - Growing items.chromium.png|0| Added|
|Stack.Horizontal Stack - Horizontally centered.chromium.png|0| Added|
|Stack.Horizontal Stack - Item alignments.chromium.png|0| Added|
|Stack.Horizontal Stack - Padding.chromium.png|0| Added|
|Stack.Horizontal Stack - Reversed.chromium.png|0| Added|
|Stack.Horizontal Stack - Right-aligned.chromium.png|0| Added|
|Stack.Horizontal Stack - Shrinking items.chromium.png|0| Added|
|Stack.Horizontal Stack - Space around.chromium.png|0| Added|
|Stack.Horizontal Stack - Space between.chromium.png|0| Added|
|Stack.Horizontal Stack - Space evenly.chromium.png|0| Added|
|Stack.Horizontal Stack - Vertically centered.chromium.png|0| Added|
|Stack.Horizontal Stack - Wrap with specified vertical gap.chromium.png|0| Added|
|Stack.Horizontal Stack - Wrap.chromium.png|0| Added|
|Stack.Horizontal Stack Default RTL.chromium.png|0| Added|
|Stack.Horizontal Stack Gap RTL.chromium.png|0| Added|
|Stack.Horizontal Stack Item Alignments RTL.chromium.png|0| Added|
|Stack.Horizontal Stack Reversed RTL.chromium.png|0| Added|
|Stack.Horizontal Stack Right Aligned RTL.chromium.png|0| Added|
|Stack.Horizontal Stack Wrap RTL.chromium.png|0| Added|
|Stack.Vertical Stack - Bottom-aligned.chromium.png|0| Added|
|Stack.Vertical Stack - Box shadow around items.chromium.png|0| Added|
|Stack.Vertical Stack - Default.chromium.png|0| Added|
|Stack.Vertical Stack - Gap.chromium.png|0| Added|
|Stack.Vertical Stack - Growing items.chromium.png|0| Added|
|Stack.Vertical Stack - Horizontally centered.chromium.png|0| Added|
|Stack.Vertical Stack - Item alignments.chromium.png|0| Added|
|Stack.Vertical Stack - Padding.chromium.png|0| Added|
|Stack.Vertical Stack - Reversed.chromium.png|0| Added|
|Stack.Vertical Stack - Right-aligned.chromium.png|0| Added|
|Stack.Vertical Stack - Shrinking items.chromium.png|0| Added|
|Stack.Vertical Stack - Space around.chromium.png|0| Added|
|Stack.Vertical Stack - Space between.chromium.png|0| Added|
|Stack.Vertical Stack - Space evenly.chromium.png|0| Added|
|Stack.Vertical Stack - Vertically centered.chromium.png|0| Added|
|Stack.Vertical Stack - Wrap.chromium.png|0| Added|
|Stack.Vertical Stack Default RTL.chromium.png|0| Added|
|Stack.Vertical Stack Item Alignments RTL.chromium.png|0| Added|
|Stack.Vertical Stack Reversed RTL.chromium.png|0| Added|
|Stack.Vertical Stack Right Aligned RTL.chromium.png|0| Added|
|Stack.Vertical Stack Wrap RTL.chromium.png|0| Added|
|Stack.Horizontal Stack - Bottom-aligned.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Box shadow around items.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Default - RTL.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Default.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Gap - RTL.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Gap.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Growing items.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Horizontally centered.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Item alignments - RTL.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Item alignments.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Padding.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Reversed - RTL.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Reversed.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Right-aligned - RTL.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Right-aligned.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Shrinking items.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Space around.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Space between.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Space evenly.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Vertically centered.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Wrap - RTL.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Wrap with specified vertical gap.default.chromium.png|0| Removed|
|Stack.Horizontal Stack - Wrap.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Bottom-aligned.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Box shadow around items.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Default - RTL.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Default.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Gap.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Growing items.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Horizontally centered.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Item alignments - RTL.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Item alignments.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Padding.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Reversed - RTL.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Reversed.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Right-aligned - RTL.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Right-aligned.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Shrinking items.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Space around.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Space between.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Space evenly.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Vertically centered.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Wrap - RTL.default.chromium.png|0| Removed|
|Stack.Vertical Stack - Wrap.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Sticky breadcrumb and sticky details list header</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.chromium.png|0| Added|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.default.chromium.png|0| Removed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll back to the top when horizontal scroll is non-zero.chromium.png|0| Removed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down by a small amount so that the first row is still visible and the header becomes sticky.chromium.png|0| Removed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down to the bottom.chromium.png|0| Removed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down when horizontal scroll is non-zero.chromium.png|0| Removed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll right.chromium.png|0| Removed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll up to the top.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchColorPicker</b> 49 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchColorPicker.Circle RTL.chromium.png|0| Added|
|SwatchColorPicker.Circle over 24px size.chromium.png|0| Added|
|SwatchColorPicker.Circle.chromium.png|0| Added|
|SwatchColorPicker.Disabled.chromium.png|0| Added|
|SwatchColorPicker.Multiple rows.chromium.png|0| Added|
|SwatchColorPicker.Square over 24px size.chromium.png|0| Added|
|SwatchColorPicker.Square.chromium.png|0| Added|
|SwatchColorPicker.Circle - RTL.Focused.chromium.png|0| Removed|
|SwatchColorPicker.Circle - RTL.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Circle - RTL.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Circle - RTL.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Circle - RTL.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Circle - RTL.default.chromium.png|0| Removed|
|SwatchColorPicker.Circle over 24px size.Focused.chromium.png|0| Removed|
|SwatchColorPicker.Circle over 24px size.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Circle over 24px size.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Circle over 24px size.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Circle over 24px size.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Circle over 24px size.default.chromium.png|0| Removed|
|SwatchColorPicker.Circle.Focused.chromium.png|0| Removed|
|SwatchColorPicker.Circle.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Circle.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Circle.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Circle.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Circle.default.chromium.png|0| Removed|
|SwatchColorPicker.Disabled.Focused.chromium.png|0| Removed|
|SwatchColorPicker.Disabled.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Disabled.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Disabled.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Disabled.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Disabled.default.chromium.png|0| Removed|
|SwatchColorPicker.Multiple rows.Focused.chromium.png|0| Removed|
|SwatchColorPicker.Multiple rows.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Multiple rows.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Multiple rows.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Multiple rows.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Multiple rows.default.chromium.png|0| Removed|
|SwatchColorPicker.Square over 24px size.Focused.chromium.png|0| Removed|
|SwatchColorPicker.Square over 24px size.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Square over 24px size.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Square over 24px size.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Square over 24px size.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Square over 24px size.default.chromium.png|0| Removed|
|SwatchColorPicker.Square.Focused.chromium.png|0| Removed|
|SwatchColorPicker.Square.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Square.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Square.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Square.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Square.default.chromium.png|0| Removed|

</details>


  <details><summary><b>TagItem</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagItem.Selected RTL.chromium.png|0| Added|
|TagItem.Selected.chromium.png|0| Added|
|TagItem.Selected - RTL.Tag Item Clear Button Hover.chromium.png|0| Removed|
|TagItem.Selected - RTL.Tag Item Clear Button Selected Hover.chromium.png|0| Removed|
|TagItem.Selected - RTL.Tag Item Hover.chromium.png|0| Removed|
|TagItem.Selected - RTL.Tag Item Select.chromium.png|0| Removed|
|TagItem.Selected - RTL.default.chromium.png|0| Removed|
|TagItem.Selected.Tag Item Clear Button Hover.chromium.png|0| Removed|
|TagItem.Selected.Tag Item Clear Button Selected Hover.chromium.png|0| Removed|
|TagItem.Selected.Tag Item Hover.chromium.png|0| Removed|
|TagItem.Selected.Tag Item Select.chromium.png|0| Removed|
|TagItem.Selected.default.chromium.png|0| Removed|

</details>


  <details><summary><b>TagPicker</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.Root.chromium.png|0| Added|
|TagPicker.Selected RTL.chromium.png|0| Added|
|TagPicker.Selected.chromium.png|0| Added|
|TagPicker.With long tag.chromium.png|0| Added|
|TagPicker.Root.Open Suggestion Menu.chromium.png|0| Removed|
|TagPicker.Root.Suggestion Menu Item Hover.chromium.png|0| Removed|
|TagPicker.Root.default.chromium.png|0| Removed|
|TagPicker.Selected - RTL.Open Suggestion Menu.chromium.png|0| Removed|
|TagPicker.Selected - RTL.Suggestion Menu Item Hover.chromium.png|0| Removed|
|TagPicker.Selected - RTL.default.chromium.png|0| Removed|
|TagPicker.Selected.Open Suggestion Menu.chromium.png|0| Removed|
|TagPicker.Selected.Suggestion Menu Item Hover.chromium.png|0| Removed|
|TagPicker.Selected.default.chromium.png|0| Removed|
|TagPicker.With long tag.default.chromium.png|0| Removed|

</details>


  <details><summary><b>TeachingBubble</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TeachingBubble.SmallHeadline.chromium.png|0| Added|
|TeachingBubble.Wide Illustration RTL.chromium.png|0| Added|
|TeachingBubble.WideIllustration.chromium.png|0| Added|
|TeachingBubble.SmallHeadline.default.chromium.png|0| Removed|
|TeachingBubble.WideIllustration - RTL.default.chromium.png|0| Removed|
|TeachingBubble.WideIllustration.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Text</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.chromium.png|0| Added|
|Text.No wrap.chromium.png|0| Added|
|Text.Root RTL.chromium.png|0| Added|
|Text.Root.chromium.png|0| Added|
|Text.Wrap.chromium.png|0| Added|
|Text.Block.default.chromium.png|0| Removed|
|Text.No wrap.default.chromium.png|0| Removed|
|Text.Root - RTL.default.chromium.png|0| Removed|
|Text.Root.default.chromium.png|0| Removed|
|Text.Wrap.default.chromium.png|0| Removed|

</details>


  <details><summary><b>TextField</b> 80 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextField.Borderless.chromium.png|0| Added|
|TextField.Disabled.chromium.png|0| Added|
|TextField.Error RTL.chromium.png|0| Added|
|TextField.Error.chromium.png|0| Added|
|TextField.Icon RTL.chromium.png|0| Added|
|TextField.Icon.chromium.png|0| Added|
|TextField.Multiline RTL.chromium.png|0| Added|
|TextField.Multiline nonresizable.chromium.png|0| Added|
|TextField.Multiline.chromium.png|0| Added|
|TextField.Placeholder RTL.chromium.png|0| Added|
|TextField.Placeholder.chromium.png|0| Added|
|TextField.Prefix With Value Disabled And Prefix Style Override RTL.chromium.png|0| Added|
|TextField.Prefix With Value Disabled RTL.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.chromium.png|0| Added|
|TextField.Prefix with Value, Disabled.chromium.png|0| Added|
|TextField.Required.chromium.png|0| Added|
|TextField.Root.chromium.png|0| Added|
|TextField.Suffix RTL.chromium.png|0| Added|
|TextField.Suffix.chromium.png|0| Added|
|TextField.Underlined.chromium.png|0| Added|
|TextField.Borderless.click.chromium.png|0| Removed|
|TextField.Borderless.default.chromium.png|0| Removed|
|TextField.Borderless.hover.chromium.png|0| Removed|
|TextField.Disabled.click.chromium.png|0| Removed|
|TextField.Disabled.default.chromium.png|0| Removed|
|TextField.Disabled.hover.chromium.png|0| Removed|
|TextField.Error - RTL.click.chromium.png|0| Removed|
|TextField.Error - RTL.default.chromium.png|0| Removed|
|TextField.Error - RTL.hover.chromium.png|0| Removed|
|TextField.Error.click.chromium.png|0| Removed|
|TextField.Error.default.chromium.png|0| Removed|
|TextField.Error.hover.chromium.png|0| Removed|
|TextField.Icon - RTL.click.chromium.png|0| Removed|
|TextField.Icon - RTL.default.chromium.png|0| Removed|
|TextField.Icon - RTL.hover.chromium.png|0| Removed|
|TextField.Icon.click.chromium.png|0| Removed|
|TextField.Icon.default.chromium.png|0| Removed|
|TextField.Icon.hover.chromium.png|0| Removed|
|TextField.Multiline - RTL.click.chromium.png|0| Removed|
|TextField.Multiline - RTL.default.chromium.png|0| Removed|
|TextField.Multiline - RTL.hover.chromium.png|0| Removed|
|TextField.Multiline nonresizable.click.chromium.png|0| Removed|
|TextField.Multiline nonresizable.default.chromium.png|0| Removed|
|TextField.Multiline nonresizable.hover.chromium.png|0| Removed|
|TextField.Multiline.click.chromium.png|0| Removed|
|TextField.Multiline.default.chromium.png|0| Removed|
|TextField.Multiline.hover.chromium.png|0| Removed|
|TextField.Placeholder - RTL.click.chromium.png|0| Removed|
|TextField.Placeholder - RTL.default.chromium.png|0| Removed|
|TextField.Placeholder - RTL.hover.chromium.png|0| Removed|
|TextField.Placeholder.click.chromium.png|0| Removed|
|TextField.Placeholder.default.chromium.png|0| Removed|
|TextField.Placeholder.hover.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled - RTL.click.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled - RTL.default.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled - RTL.hover.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.click.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.default.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.hover.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.click.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.default.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.hover.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled.click.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled.default.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled.hover.chromium.png|0| Removed|
|TextField.Required.click.chromium.png|0| Removed|
|TextField.Required.default.chromium.png|0| Removed|
|TextField.Required.hover.chromium.png|0| Removed|
|TextField.Root.click.chromium.png|0| Removed|
|TextField.Root.default.chromium.png|0| Removed|
|TextField.Root.hover.chromium.png|0| Removed|
|TextField.Suffix - RTL.click.chromium.png|0| Removed|
|TextField.Suffix - RTL.default.chromium.png|0| Removed|
|TextField.Suffix - RTL.hover.chromium.png|0| Removed|
|TextField.Suffix.click.chromium.png|0| Removed|
|TextField.Suffix.default.chromium.png|0| Removed|
|TextField.Suffix.hover.chromium.png|0| Removed|
|TextField.Underlined.click.chromium.png|0| Removed|
|TextField.Underlined.default.chromium.png|0| Removed|
|TextField.Underlined.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>ThemeProvider</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ThemeProvider.Customized styles.chromium.png|0| Added|
|ThemeProvider.Customized theme - nested ThemeProvider.chromium.png|0| Added|
|ThemeProvider.Customized theme.chromium.png|0| Added|
|ThemeProvider.Default theme.chromium.png|0| Added|
|ThemeProvider.Customized styles.default.chromium.png|0| Removed|
|ThemeProvider.Customized theme - nested ThemeProvider.default.chromium.png|0| Removed|
|ThemeProvider.Customized theme.default.chromium.png|0| Removed|
|ThemeProvider.Default theme.default.chromium.png|0| Removed|

</details>


  <details><summary><b>ThemeProvider with Customizer</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ThemeProvider with Customizer.Customizer wraps ThemeProvider.chromium.png|0| Added|
|ThemeProvider with Customizer.ThemeProvider wraps Customizer.chromium.png|0| Added|
|ThemeProvider with Customizer.Customizer wraps ThemeProvider.default.chromium.png|0| Removed|
|ThemeProvider with Customizer.ThemeProvider wraps Customizer.default.chromium.png|0| Removed|

</details>


  <details><summary><b>ThemeProvider with loadTheme</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ThemeProvider with loadTheme.Use contextual theme over global theme if defined.chromium.png|0| Added|
|ThemeProvider with loadTheme.Use updated global theme.chromium.png|0| Added|
|ThemeProvider with loadTheme.Use contextual theme over global theme if defined.default.chromium.png|0| Removed|
|ThemeProvider with loadTheme.Use contextual theme over global theme if defined.global theme changed.chromium.png|0| Removed|
|ThemeProvider with loadTheme.Use updated global theme.default.chromium.png|0| Removed|
|ThemeProvider with loadTheme.Use updated global theme.global theme changed.chromium.png|0| Removed|

</details>


  <details><summary><b>Tile</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tile.Document tile with fit landscape image.chromium.png|0| Added|
|Tile.Document tile with fit portrait image.chromium.png|0| Added|
|Tile.Document tile with icon-sized image.chromium.png|0| Added|
|Tile.Document tile with icon.chromium.png|0| Added|
|Tile.Tile with no content and long text.chromium.png|0| Added|
|Tile.Document tile with fit landscape image.default.chromium.png|0| Removed|
|Tile.Document tile with fit portrait image.default.chromium.png|0| Removed|
|Tile.Document tile with icon-sized image.default.chromium.png|0| Removed|
|Tile.Document tile with icon.default.chromium.png|0| Removed|
|Tile.Tile with no content and long text.default.chromium.png|0| Removed|

</details>


  <details><summary><b>TilesList</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TilesList.Basic RTL.chromium.png|0| Added|
|TilesList.Basic.chromium.png|0| Added|
|TilesList.Basic - RTL.default.chromium.png|0| Removed|
|TilesList.Basic.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Toggle</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toggle.Checked RTL.chromium.png|0| Added|
|Toggle.Checked.chromium.png|0| Added|
|Toggle.Disabled checked.chromium.png|0| Added|
|Toggle.Disabled unchecked.chromium.png|0| Added|
|Toggle.Unchecked RTL.chromium.png|0| Added|
|Toggle.Unchecked.chromium.png|0| Added|
|Toggle.With inline label (JSX Element) and without onText and offText.chromium.png|0| Added|
|Toggle.With inline label (JSX Element).chromium.png|0| Added|
|Toggle.With inline label and without onText and offText.chromium.png|0| Added|
|Toggle.With inline label disabled and without onText and offText.chromium.png|0| Added|
|Toggle.With inline label disabled.chromium.png|0| Added|
|Toggle.With inline label.chromium.png|0| Added|
|Toggle.Checked - RTL.default.chromium.png|0| Removed|
|Toggle.Checked.default.chromium.png|0| Removed|
|Toggle.Disabled checked.default.chromium.png|0| Removed|
|Toggle.Disabled unchecked.default.chromium.png|0| Removed|
|Toggle.Unchecked - RTL.default.chromium.png|0| Removed|
|Toggle.Unchecked.default.chromium.png|0| Removed|
|Toggle.With inline label (JSX Element) and without onText and offText.default.chromium.png|0| Removed|
|Toggle.With inline label (JSX Element).default.chromium.png|0| Removed|
|Toggle.With inline label and without onText and offText.default.chromium.png|0| Removed|
|Toggle.With inline label disabled and without onText and offText.default.chromium.png|0| Removed|
|Toggle.With inline label disabled.default.chromium.png|0| Removed|
|Toggle.With inline label.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Tooltip</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip.Default.chromium.png|1613354| Changed|
|Tooltip.Default.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Tooltip - Multiple</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip - Multiple.Two Tooltips.chromium.png|1612482| Changed|
|Tooltip - Multiple.Two Tooltips.hover inner.chromium.png|0| Removed|
|Tooltip - Multiple.Two Tooltips.hover outer.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 22 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Basic Dark Mode.chromium.png|0| Added|
|react-charting-AreaChart.Basic RTL.chromium.png|0| Added|
|react-charting-AreaChart.Basic.chromium.png|0| Added|
|react-charting-AreaChart.Custom Accessibility Dark Mode.chromium.png|0| Added|
|react-charting-AreaChart.Custom Accessibility RTL.chromium.png|0| Added|
|react-charting-AreaChart.Custom Accessibility.chromium.png|0| Added|
|react-charting-AreaChart.Multiple Dark Mode.chromium.png|0| Added|
|react-charting-AreaChart.Multiple RTL.chromium.png|0| Added|
|react-charting-AreaChart.Multiple.chromium.png|0| Added|
|react-charting-AreaChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-AreaChart.Basic - Dark Mode.hover.chromium.png|0| Removed|
|react-charting-AreaChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-AreaChart.Basic.default.chromium.png|0| Removed|
|react-charting-AreaChart.Basic.hover.chromium.png|0| Removed|
|react-charting-AreaChart.Custom Accessibility - Dark Mode.default.chromium.png|0| Removed|
|react-charting-AreaChart.Custom Accessibility - RTL.default.chromium.png|0| Removed|
|react-charting-AreaChart.Custom Accessibility.default.chromium.png|0| Removed|
|react-charting-AreaChart.Multiple - Dark Mode.default.chromium.png|0| Removed|
|react-charting-AreaChart.Multiple - Dark Mode.hover.chromium.png|0| Removed|
|react-charting-AreaChart.Multiple - RTL.default.chromium.png|0| Removed|
|react-charting-AreaChart.Multiple.default.chromium.png|0| Removed|
|react-charting-AreaChart.Multiple.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-DonutChart</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-DonutChart.Basic Dark Mode.chromium.png|0| Added|
|react-charting-DonutChart.Basic RTL.chromium.png|0| Added|
|react-charting-DonutChart.Basic.chromium.png|0| Added|
|react-charting-DonutChart.Dynamic Dark Mode.chromium.png|0| Added|
|react-charting-DonutChart.Dynamic RTL.chromium.png|0| Added|
|react-charting-DonutChart.Dynamic.chromium.png|0| Added|
|react-charting-DonutChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-DonutChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-DonutChart.Basic.default.chromium.png|0| Removed|
|react-charting-DonutChart.Dynamic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-DonutChart.Dynamic - RTL.default.chromium.png|0| Removed|
|react-charting-DonutChart.Dynamic.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-GaugeChart</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-GaugeChart.Basic Dark Mode.chromium.png|0| Added|
|react-charting-GaugeChart.Basic RTL.chromium.png|0| Added|
|react-charting-GaugeChart.Basic.chromium.png|0| Added|
|react-charting-GaugeChart.Variant Dark Mode.chromium.png|0| Added|
|react-charting-GaugeChart.Variant RTL.chromium.png|0| Added|
|react-charting-GaugeChart.Variant.chromium.png|0| Added|
|react-charting-GaugeChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-GaugeChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-GaugeChart.Basic.default.chromium.png|0| Removed|
|react-charting-GaugeChart.Variant - Dark Mode.default.chromium.png|0| Removed|
|react-charting-GaugeChart.Variant - RTL.default.chromium.png|0| Removed|
|react-charting-GaugeChart.Variant.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic Dark Mode.chromium.png|0| Added|
|react-charting-HeatMapChart.Basic RTL.chromium.png|0| Added|
|react-charting-HeatMapChart.Basic.chromium.png|0| Added|
|react-charting-HeatMapChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-HeatMapChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-HeatMapChart.Basic.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-HorizontalBarChart</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HorizontalBarChart.Basic Dark Mode.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Basic RTL.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Basic.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Variant Dark Mode.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Variant RTL.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Variant.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With Axis Dark Mode.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With Axis RTL.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With Benchmark Dark Mode.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With Benchmark RTL.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With_Axis.chromium.png|0| Added|
|react-charting-HorizontalBarChart.With_Benchmark.chromium.png|0| Added|
|react-charting-HorizontalBarChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.Basic - Dark Mode.hover.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.Basic.default.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.Basic.hover.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.Variant - Dark Mode.default.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.Variant - RTL.default.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.Variant.default.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.With_Axis - Dark Mode.default.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.With_Axis - RTL.default.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.With_Axis.default.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.With_Benchmark - Dark Mode.default.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.With_Benchmark - RTL.default.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.With_Benchmark.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-Legend</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-Legend.Basic Dark Mode.chromium.png|0| Added|
|react-charting-Legend.Basic RTL.chromium.png|0| Added|
|react-charting-Legend.Basic.chromium.png|0| Added|
|react-charting-Legend.Overflow Dark Mode.chromium.png|0| Added|
|react-charting-Legend.Overflow RTL.chromium.png|0| Added|
|react-charting-Legend.Overflow.chromium.png|0| Added|
|react-charting-Legend.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-Legend.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-Legend.Basic.default.chromium.png|0| Removed|
|react-charting-Legend.Overflow - Dark Mode.default.chromium.png|0| Removed|
|react-charting-Legend.Overflow - Dark Mode.expanded.chromium.png|0| Removed|
|react-charting-Legend.Overflow - RTL.default.chromium.png|0| Removed|
|react-charting-Legend.Overflow - RTL.expanded.chromium.png|0| Removed|
|react-charting-Legend.Overflow.default.chromium.png|0| Removed|
|react-charting-Legend.Overflow.expanded.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-LineChart</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Basic Dark Mode.chromium.png|0| Added|
|react-charting-LineChart.Basic RTL.chromium.png|0| Added|
|react-charting-LineChart.Basic.chromium.png|0| Added|
|react-charting-LineChart.Events Dark Mode.chromium.png|0| Added|
|react-charting-LineChart.Events RTL.chromium.png|0| Added|
|react-charting-LineChart.Events.chromium.png|0| Added|
|react-charting-LineChart.Gaps Dark Mode.chromium.png|0| Added|
|react-charting-LineChart.Gaps RTL.chromium.png|0| Added|
|react-charting-LineChart.Gaps.chromium.png|0| Added|
|react-charting-LineChart.Multiple Dark Mode.chromium.png|0| Added|
|react-charting-LineChart.Multiple RTL.chromium.png|0| Added|
|react-charting-LineChart.Multiple.chromium.png|0| Added|
|react-charting-LineChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-LineChart.Basic - Dark Mode.hover.chromium.png|0| Removed|
|react-charting-LineChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-LineChart.Basic.default.chromium.png|0| Removed|
|react-charting-LineChart.Basic.hover.chromium.png|0| Removed|
|react-charting-LineChart.Events - Dark Mode.default.chromium.png|0| Removed|
|react-charting-LineChart.Events - RTL.default.chromium.png|0| Removed|
|react-charting-LineChart.Events.default.chromium.png|0| Removed|
|react-charting-LineChart.Gaps - Dark Mode.default.chromium.png|0| Removed|
|react-charting-LineChart.Gaps - RTL.default.chromium.png|0| Removed|
|react-charting-LineChart.Gaps.default.chromium.png|0| Removed|
|react-charting-LineChart.Multiple - Dark Mode.default.chromium.png|0| Removed|
|react-charting-LineChart.Multiple - RTL.default.chromium.png|0| Removed|
|react-charting-LineChart.Multiple.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-MultiStackBarChart</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-MultiStackBarChart.Basic Absolute Dark Mode.chromium.png|0| Added|
|react-charting-MultiStackBarChart.Basic Absolute RTL.chromium.png|0| Added|
|react-charting-MultiStackBarChart.Basic Part To Whole Dark Mode.chromium.png|0| Added|
|react-charting-MultiStackBarChart.Basic Part To Whole RTL.chromium.png|0| Added|
|react-charting-MultiStackBarChart.Basic_Absolute.chromium.png|0| Added|
|react-charting-MultiStackBarChart.Basic_PartToWhole.chromium.png|0| Added|
|react-charting-MultiStackBarChart.With Placeholders Dark Mode.chromium.png|0| Added|
|react-charting-MultiStackBarChart.With Placeholders RTL.chromium.png|0| Added|
|react-charting-MultiStackBarChart.With_Placeholders.chromium.png|0| Added|
|react-charting-MultiStackBarChart.Basic_Absolute - Dark Mode.default.chromium.png|0| Removed|
|react-charting-MultiStackBarChart.Basic_Absolute - RTL.default.chromium.png|0| Removed|
|react-charting-MultiStackBarChart.Basic_Absolute.default.chromium.png|0| Removed|
|react-charting-MultiStackBarChart.Basic_PartToWhole - Dark Mode.default.chromium.png|0| Removed|
|react-charting-MultiStackBarChart.Basic_PartToWhole - RTL.default.chromium.png|0| Removed|
|react-charting-MultiStackBarChart.Basic_PartToWhole.default.chromium.png|0| Removed|
|react-charting-MultiStackBarChart.With_Placeholders - Dark Mode.default.chromium.png|0| Removed|
|react-charting-MultiStackBarChart.With_Placeholders - RTL.default.chromium.png|0| Removed|
|react-charting-MultiStackBarChart.With_Placeholders.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-PieChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-PieChart.Basic Dark Mode.chromium.png|0| Added|
|react-charting-PieChart.Basic RTL.chromium.png|0| Added|
|react-charting-PieChart.Basic.chromium.png|0| Added|
|react-charting-PieChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-PieChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-PieChart.Basic.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-SankeyChart</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SankeyChart.Basic Dark Mode.chromium.png|0| Added|
|react-charting-SankeyChart.Basic RTL.chromium.png|0| Added|
|react-charting-SankeyChart.Basic.chromium.png|0| Added|
|react-charting-SankeyChart.Place Holder Dark Mode.chromium.png|0| Added|
|react-charting-SankeyChart.Place Holder RTL.chromium.png|0| Added|
|react-charting-SankeyChart.PlaceHolder.chromium.png|0| Added|
|react-charting-SankeyChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-SankeyChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-SankeyChart.Basic.default.chromium.png|0| Removed|
|react-charting-SankeyChart.PlaceHolder - Dark Mode.default.chromium.png|0| Removed|
|react-charting-SankeyChart.PlaceHolder - RTL.default.chromium.png|0| Removed|
|react-charting-SankeyChart.PlaceHolder.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-SparkLineChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SparkLineChart.Basic Dark Mode.chromium.png|0| Added|
|react-charting-SparkLineChart.Basic RTL.chromium.png|0| Added|
|react-charting-SparkLineChart.Basic.chromium.png|0| Added|
|react-charting-SparkLineChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-SparkLineChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-SparkLineChart.Basic.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-StackedBarChart</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-StackedBarChart.Basic Dark Mode.chromium.png|0| Added|
|react-charting-StackedBarChart.Basic RTL.chromium.png|0| Added|
|react-charting-StackedBarChart.Basic.chromium.png|0| Added|
|react-charting-StackedBarChart.Benchmark Dark Mode.chromium.png|0| Added|
|react-charting-StackedBarChart.Benchmark RTL.chromium.png|0| Added|
|react-charting-StackedBarChart.Benchmark.chromium.png|0| Added|
|react-charting-StackedBarChart.Multiple Dark Mode.chromium.png|0| Added|
|react-charting-StackedBarChart.Multiple RTL.chromium.png|0| Added|
|react-charting-StackedBarChart.Multiple.chromium.png|0| Added|
|react-charting-StackedBarChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-StackedBarChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-StackedBarChart.Basic.default.chromium.png|0| Removed|
|react-charting-StackedBarChart.Benchmark - Dark Mode.default.chromium.png|0| Removed|
|react-charting-StackedBarChart.Benchmark - RTL.default.chromium.png|0| Removed|
|react-charting-StackedBarChart.Benchmark.default.chromium.png|0| Removed|
|react-charting-StackedBarChart.Multiple - Dark Mode.default.chromium.png|0| Removed|
|react-charting-StackedBarChart.Multiple - RTL.default.chromium.png|0| Removed|
|react-charting-StackedBarChart.Multiple.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-TreeChart</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-TreeChart.Basic Dark Mode.chromium.png|0| Added|
|react-charting-TreeChart.Basic RTL.chromium.png|0| Added|
|react-charting-TreeChart.Basic.chromium.png|0| Added|
|react-charting-TreeChart.Compact Dark Mode.chromium.png|0| Added|
|react-charting-TreeChart.Compact RTL.chromium.png|0| Added|
|react-charting-TreeChart.Compact.chromium.png|0| Added|
|react-charting-TreeChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-TreeChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-TreeChart.Basic.default.chromium.png|0| Removed|
|react-charting-TreeChart.Compact - Dark Mode.default.chromium.png|0| Removed|
|react-charting-TreeChart.Compact - RTL.default.chromium.png|0| Removed|
|react-charting-TreeChart.Compact.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 42 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|0| Added|
|react-charting-VerticalBarChart.Basic Secondary Y Axis Dark Mode.chromium.png|0| Added|
|react-charting-VerticalBarChart.Basic Secondary Y Axis RTL.chromium.png|0| Added|
|react-charting-VerticalBarChart.Date Axis Vbc Dark Mode.chromium.png|0| Added|
|react-charting-VerticalBarChart.Date Axis Vbc RTL.chromium.png|0| Added|
|react-charting-VerticalBarChart.Date Axis- VBC.chromium.png|0| Added|
|react-charting-VerticalBarChart.Dynamic - Wrap Labels.chromium.png|0| Added|
|react-charting-VerticalBarChart.Dynamic Wrap Labels Dark Mode.chromium.png|0| Added|
|react-charting-VerticalBarChart.Dynamic Wrap Labels RTL.chromium.png|0| Added|
|react-charting-VerticalBarChart.Grouped - Wrap Labels.chromium.png|0| Added|
|react-charting-VerticalBarChart.Grouped Wrap Labels Dark Mode.chromium.png|0| Added|
|react-charting-VerticalBarChart.Grouped Wrap Labels RTL.chromium.png|0| Added|
|react-charting-VerticalBarChart.Rotated Label Hide Legends Dark Mode.chromium.png|0| Added|
|react-charting-VerticalBarChart.Rotated Label Hide Legends RTL.chromium.png|0| Added|
|react-charting-VerticalBarChart.Rotated Label- Hide Legends.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked - Date Axis.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked Dark Mode.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked Date Axis Dark Mode.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked Date Axis RTL.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked RTL.chromium.png|0| Added|
|react-charting-VerticalBarChart.Stacked.chromium.png|0| Added|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis - Dark Mode.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis - RTL.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Date Axis- VBC - Dark Mode.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Date Axis- VBC - RTL.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Date Axis- VBC.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Dynamic - Wrap Labels - Dark Mode.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Dynamic - Wrap Labels - RTL.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Dynamic - Wrap Labels.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Grouped - Wrap Labels - Dark Mode.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Grouped - Wrap Labels - RTL.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Grouped - Wrap Labels.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Rotated Label- Hide Legends - Dark Mode.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Rotated Label- Hide Legends - RTL.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Rotated Label- Hide Legends.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Stacked - Dark Mode.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Stacked - Date Axis - Dark Mode.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Stacked - Date Axis - RTL.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Stacked - Date Axis.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Stacked - RTL.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Stacked.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>